
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>ECO010-megahit - 7 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="ECO010-megahit - 7 record(s) - 7 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/ECO010-megahit/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="ECO010-megahit.gbk">Download GenBank summary file</a></li>
<li><a href="ECO010-megahit.json">Download JSON results file</a></li>

<li><a href="log_ECO010-megahit.txt">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton RiPP RRE-containing r3c1">
<a href="#r3c1">3.1</a>
</div>



<div class="regbutton NRPS NRPS r15c1">
<a href="#r15c1">15.1</a>
</div>



<div class="regbutton RiPP ranthipeptide r72c1">
<a href="#r72c1">72.1</a>
</div>



<div class="regbutton other phosphonate r106c1">
<a href="#r106c1">106.1</a>
</div>



<div class="regbutton RiPP thioamitides r259c1">
<a href="#r259c1">259.1</a>
</div>



<div class="regbutton NRPS r338c1">
<a href="#r338c1">338.1</a>
</div>



<div class="regbutton NRPS NRPS-like r381c1">
<a href="#r381c1">381.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>NODE_3_length_18813_cov_105.5</strong> (None)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="NODE_3_length_18813_cov_105.5_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton RiPP RRE-containing r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">6,928</td>
<td class="digits table-split-left">18,813</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_15_length_13587_cov_90.8</strong> (None)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-15" data-filename="NODE_15_length_13587_cov_90.8_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-15" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r15c1">
<td class="regbutton NRPS NRPS r15c1">
<a href="#r15c1">Region&nbsp15.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">13,587</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_72_length_8481_cov_79.8</strong> (None)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-72" data-filename="NODE_72_length_8481_cov_79.8_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-72" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r72c1">
<td class="regbutton RiPP ranthipeptide r72c1">
<a href="#r72c1">Region&nbsp72.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ranthipeptide" target="_blank">ranthipeptide</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">8,481</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_106_length_7571_cov_86.5</strong> (None)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-106" data-filename="NODE_106_length_7571_cov_86.5_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-106" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r106c1">
<td class="regbutton other phosphonate r106c1">
<a href="#r106c1">Region&nbsp106.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phosphonate" target="_blank">phosphonate</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">7,571</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001411/1" target="_blank">polysaccharide B</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_259_length_5660_cov_107.3</strong> (None)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-259" data-filename="NODE_259_length_5660_cov_107.3_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-259" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r259c1">
<td class="regbutton RiPP thioamitides r259c1">
<a href="#r259c1">Region&nbsp259.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thioamitides" target="_blank">thioamitides</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">5,660</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_338_length_5055_cov_56.8</strong> (None)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-338" data-filename="NODE_338_length_5055_cov_56.8_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-338" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r338c1">
<td class="regbutton NRPS r338c1">
<a href="#r338c1">Region&nbsp338.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">5,055</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>NODE_381_length_4785_cov_56.0</strong> (None)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-381" data-filename="NODE_381_length_4785_cov_56.0_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-381" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r381c1">
<td class="regbutton NRPS NRPS-like r381c1">
<a href="#r381c1">Region&nbsp381.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">4,785</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton RiPP RRE-containing r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#rre-containing" target="_blank">RRE-containing</a>
</td>
<td class="digits">6,928</td>
<td class="digits table-split-left">18,813</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r15c1">
<td class="regbutton NRPS NRPS r15c1">
<a href="#r15c1">Region&nbsp15.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">13,587</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r72c1">
<td class="regbutton RiPP ranthipeptide r72c1">
<a href="#r72c1">Region&nbsp72.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ranthipeptide" target="_blank">ranthipeptide</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">8,481</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r106c1">
<td class="regbutton other phosphonate r106c1">
<a href="#r106c1">Region&nbsp106.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#phosphonate" target="_blank">phosphonate</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">7,571</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001411/1" target="_blank">polysaccharide B</a></td>
<td>Saccharide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 6%, #ffffff00 6%)">6%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r259c1">
<td class="regbutton RiPP thioamitides r259c1">
<a href="#r259c1">Region&nbsp259.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thioamitides" target="_blank">thioamitides</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">5,660</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r338c1">
<td class="regbutton NRPS r338c1">
<a href="#r338c1">Region&nbsp338.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">5,055</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r381c1">
<td class="regbutton NRPS NRPS-like r381c1">
<a href="#r381c1">Region&nbsp381.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">4,785</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>NODE_1_length_30799_cov_97.5</strong> (None)</dd>

<dd><strong>NODE_2_length_23976_cov_84.2</strong> (None)</dd>

<dd><strong>NODE_4_length_17143_cov_100.9</strong> (None)</dd>

<dd><strong>NODE_5_length_16837_cov_103.1</strong> (None)</dd>

<dd><strong>NODE_6_length_16502_cov_111.5</strong> (None)</dd>

<dd><strong>NODE_7_length_15953_cov_99.8</strong> (None)</dd>

<dd><strong>NODE_8_length_14907_cov_101.2</strong> (None)</dd>

<dd><strong>NODE_9_length_14660_cov_99.1</strong> (None)</dd>

<dd><strong>NODE_10_length_14571_cov_104.3</strong> (None)</dd>

<dd><strong>NODE_11_length_14414_cov_101.9</strong> (None)</dd>

<dd><strong>NODE_12_length_14308_cov_86.0</strong> (None)</dd>

<dd><strong>NODE_13_length_14078_cov_83.5</strong> (None)</dd>

<dd><strong>NODE_14_length_14027_cov_85.8</strong> (None)</dd>

<dd><strong>NODE_16_length_12838_cov_95.5</strong> (None)</dd>

<dd><strong>NODE_17_length_12754_cov_94.2</strong> (None)</dd>

<dd><strong>NODE_18_length_12537_cov_96.1</strong> (None)</dd>

<dd><strong>NODE_19_length_12147_cov_81.4</strong> (None)</dd>

<dd><strong>NODE_20_length_12011_cov_103.4</strong> (None)</dd>

<dd><strong>NODE_21_length_11992_cov_69.4</strong> (None)</dd>

<dd><strong>NODE_22_length_11989_cov_76.5</strong> (None)</dd>

<dd><strong>NODE_23_length_11667_cov_80.9</strong> (None)</dd>

<dd><strong>NODE_24_length_11456_cov_96.3</strong> (None)</dd>

<dd><strong>NODE_25_length_11436_cov_111.7</strong> (None)</dd>

<dd><strong>NODE_26_length_11381_cov_91.6</strong> (None)</dd>

<dd><strong>NODE_27_length_11375_cov_91.3</strong> (None)</dd>

<dd><strong>NODE_28_length_11299_cov_95.4</strong> (None)</dd>

<dd><strong>NODE_29_length_11125_cov_94.0</strong> (None)</dd>

<dd><strong>NODE_30_length_11110_cov_83.9</strong> (None)</dd>

<dd><strong>NODE_31_length_10975_cov_76.1</strong> (None)</dd>

<dd><strong>NODE_32_length_10747_cov_72.8</strong> (None)</dd>

<dd><strong>NODE_33_length_10729_cov_103.7</strong> (None)</dd>

<dd><strong>NODE_34_length_10632_cov_79.0</strong> (None)</dd>

<dd><strong>NODE_35_length_10565_cov_72.7</strong> (None)</dd>

<dd><strong>NODE_36_length_10350_cov_83.3</strong> (None)</dd>

<dd><strong>NODE_37_length_10349_cov_105.9</strong> (None)</dd>

<dd><strong>NODE_38_length_10198_cov_89.5</strong> (None)</dd>

<dd><strong>NODE_39_length_10135_cov_93.2</strong> (None)</dd>

<dd><strong>NODE_40_length_10130_cov_102.5</strong> (None)</dd>

<dd><strong>NODE_41_length_10002_cov_83.6</strong> (None)</dd>

<dd><strong>NODE_42_length_9904_cov_87.9</strong> (None)</dd>

<dd><strong>NODE_43_length_9860_cov_95.4</strong> (None)</dd>

<dd><strong>NODE_44_length_9781_cov_78.1</strong> (None)</dd>

<dd><strong>NODE_45_length_9755_cov_81.8</strong> (None)</dd>

<dd><strong>NODE_46_length_9703_cov_66.7</strong> (None)</dd>

<dd><strong>NODE_47_length_9675_cov_91.8</strong> (None)</dd>

<dd><strong>NODE_48_length_9647_cov_105.6</strong> (None)</dd>

<dd><strong>NODE_49_length_9640_cov_97.9</strong> (None)</dd>

<dd><strong>NODE_50_length_9640_cov_70.1</strong> (None)</dd>

<dd><strong>NODE_51_length_9610_cov_115.8</strong> (None)</dd>

<dd><strong>NODE_52_length_9589_cov_94.3</strong> (None)</dd>

<dd><strong>NODE_53_length_9542_cov_107.4</strong> (None)</dd>

<dd><strong>NODE_54_length_9498_cov_117.0</strong> (None)</dd>

<dd><strong>NODE_55_length_9411_cov_119.1</strong> (None)</dd>

<dd><strong>NODE_56_length_9376_cov_79.3</strong> (None)</dd>

<dd><strong>NODE_57_length_9364_cov_94.0</strong> (None)</dd>

<dd><strong>NODE_58_length_9307_cov_60.8</strong> (None)</dd>

<dd><strong>NODE_59_length_9283_cov_110.4</strong> (None)</dd>

<dd><strong>NODE_60_length_9265_cov_87.4</strong> (None)</dd>

<dd><strong>NODE_61_length_9235_cov_105.3</strong> (None)</dd>

<dd><strong>NODE_62_length_9225_cov_78.0</strong> (None)</dd>

<dd><strong>NODE_63_length_9191_cov_79.8</strong> (None)</dd>

<dd><strong>NODE_64_length_9070_cov_83.5</strong> (None)</dd>

<dd><strong>NODE_65_length_9070_cov_96.5</strong> (None)</dd>

<dd><strong>NODE_66_length_8982_cov_95.1</strong> (None)</dd>

<dd><strong>NODE_67_length_8884_cov_59.9</strong> (None)</dd>

<dd><strong>NODE_68_length_8778_cov_94.6</strong> (None)</dd>

<dd><strong>NODE_69_length_8771_cov_101.9</strong> (None)</dd>

<dd><strong>NODE_70_length_8653_cov_86.5</strong> (None)</dd>

<dd><strong>NODE_71_length_8505_cov_82.9</strong> (None)</dd>

<dd><strong>NODE_73_length_8475_cov_99.1</strong> (None)</dd>

<dd><strong>NODE_74_length_8447_cov_77.0</strong> (None)</dd>

<dd><strong>NODE_75_length_8438_cov_73.6</strong> (None)</dd>

<dd><strong>NODE_76_length_8425_cov_84.3</strong> (None)</dd>

<dd><strong>NODE_77_length_8414_cov_93.5</strong> (None)</dd>

<dd><strong>NODE_78_length_8355_cov_83.1</strong> (None)</dd>

<dd><strong>NODE_79_length_8338_cov_68.0</strong> (None)</dd>

<dd><strong>NODE_80_length_8335_cov_85.4</strong> (None)</dd>

<dd><strong>NODE_81_length_8306_cov_84.7</strong> (None)</dd>

<dd><strong>NODE_82_length_8288_cov_82.4</strong> (None)</dd>

<dd><strong>NODE_83_length_8230_cov_85.2</strong> (None)</dd>

<dd><strong>NODE_84_length_8211_cov_65.2</strong> (None)</dd>

<dd><strong>NODE_85_length_8155_cov_76.2</strong> (None)</dd>

<dd><strong>NODE_86_length_8154_cov_82.3</strong> (None)</dd>

<dd><strong>NODE_87_length_8126_cov_80.9</strong> (None)</dd>

<dd><strong>NODE_88_length_8104_cov_76.8</strong> (None)</dd>

<dd><strong>NODE_89_length_8039_cov_102.5</strong> (None)</dd>

<dd><strong>NODE_90_length_8034_cov_101.2</strong> (None)</dd>

<dd><strong>NODE_91_length_7984_cov_85.4</strong> (None)</dd>

<dd><strong>NODE_92_length_7958_cov_87.9</strong> (None)</dd>

<dd><strong>NODE_93_length_7877_cov_100.6</strong> (None)</dd>

<dd><strong>NODE_94_length_7876_cov_63.7</strong> (None)</dd>

<dd><strong>NODE_95_length_7846_cov_62.1</strong> (None)</dd>

<dd><strong>NODE_96_length_7764_cov_98.8</strong> (None)</dd>

<dd><strong>NODE_97_length_7731_cov_83.2</strong> (None)</dd>

<dd><strong>NODE_98_length_7727_cov_72.8</strong> (None)</dd>

<dd><strong>NODE_99_length_7687_cov_64.6</strong> (None)</dd>

<dd><strong>NODE_100_length_7657_cov_90.0</strong> (None)</dd>

<dd><strong>NODE_101_length_7655_cov_104.2</strong> (None)</dd>

<dd><strong>NODE_102_length_7653_cov_81.5</strong> (None)</dd>

<dd><strong>NODE_103_length_7635_cov_84.4</strong> (None)</dd>

<dd><strong>NODE_104_length_7594_cov_98.5</strong> (None)</dd>

<dd><strong>NODE_105_length_7576_cov_106.0</strong> (None)</dd>

<dd><strong>NODE_107_length_7546_cov_83.3</strong> (None)</dd>

<dd><strong>NODE_108_length_7499_cov_60.6</strong> (None)</dd>

<dd><strong>NODE_109_length_7489_cov_81.3</strong> (None)</dd>

<dd><strong>NODE_110_length_7487_cov_78.6</strong> (None)</dd>

<dd><strong>NODE_111_length_7478_cov_70.8</strong> (None)</dd>

<dd><strong>NODE_112_length_7472_cov_111.1</strong> (None)</dd>

<dd><strong>NODE_113_length_7471_cov_110.8</strong> (None)</dd>

<dd><strong>NODE_114_length_7432_cov_76.8</strong> (None)</dd>

<dd><strong>NODE_115_length_7430_cov_103.9</strong> (None)</dd>

<dd><strong>NODE_116_length_7419_cov_83.7</strong> (None)</dd>

<dd><strong>NODE_117_length_7392_cov_82.8</strong> (None)</dd>

<dd><strong>NODE_118_length_7372_cov_95.3</strong> (None)</dd>

<dd><strong>NODE_119_length_7341_cov_96.5</strong> (None)</dd>

<dd><strong>NODE_120_length_7338_cov_87.1</strong> (None)</dd>

<dd><strong>NODE_121_length_7326_cov_91.5</strong> (None)</dd>

<dd><strong>NODE_122_length_7270_cov_59.9</strong> (None)</dd>

<dd><strong>NODE_123_length_7254_cov_93.7</strong> (None)</dd>

<dd><strong>NODE_124_length_7243_cov_64.0</strong> (None)</dd>

<dd><strong>NODE_125_length_7218_cov_79.7</strong> (None)</dd>

<dd><strong>NODE_126_length_7179_cov_141.3</strong> (None)</dd>

<dd><strong>NODE_127_length_7177_cov_80.7</strong> (None)</dd>

<dd><strong>NODE_128_length_7175_cov_81.6</strong> (None)</dd>

<dd><strong>NODE_129_length_7160_cov_76.4</strong> (None)</dd>

<dd><strong>NODE_130_length_7149_cov_64.6</strong> (None)</dd>

<dd><strong>NODE_131_length_7142_cov_83.3</strong> (None)</dd>

<dd><strong>NODE_132_length_7130_cov_108.0</strong> (None)</dd>

<dd><strong>NODE_133_length_7129_cov_106.0</strong> (None)</dd>

<dd><strong>NODE_134_length_7123_cov_77.3</strong> (None)</dd>

<dd><strong>NODE_135_length_7105_cov_73.4</strong> (None)</dd>

<dd><strong>NODE_136_length_7090_cov_107.7</strong> (None)</dd>

<dd><strong>NODE_137_length_7090_cov_71.2</strong> (None)</dd>

<dd><strong>NODE_138_length_7082_cov_54.6</strong> (None)</dd>

<dd><strong>NODE_139_length_7076_cov_82.7</strong> (None)</dd>

<dd><strong>NODE_140_length_7041_cov_76.5</strong> (None)</dd>

<dd><strong>NODE_141_length_7041_cov_84.3</strong> (None)</dd>

<dd><strong>NODE_142_length_7032_cov_58.6</strong> (None)</dd>

<dd><strong>NODE_143_length_7031_cov_74.6</strong> (None)</dd>

<dd><strong>NODE_144_length_7020_cov_82.2</strong> (None)</dd>

<dd><strong>NODE_145_length_7016_cov_91.4</strong> (None)</dd>

<dd><strong>NODE_146_length_6979_cov_115.6</strong> (None)</dd>

<dd><strong>NODE_147_length_6941_cov_89.9</strong> (None)</dd>

<dd><strong>NODE_148_length_6921_cov_72.9</strong> (None)</dd>

<dd><strong>NODE_149_length_6872_cov_111.8</strong> (None)</dd>

<dd><strong>NODE_150_length_6840_cov_100.0</strong> (None)</dd>

<dd><strong>NODE_151_length_6839_cov_100.3</strong> (None)</dd>

<dd><strong>NODE_152_length_6833_cov_97.6</strong> (None)</dd>

<dd><strong>NODE_153_length_6826_cov_82.1</strong> (None)</dd>

<dd><strong>NODE_154_length_6819_cov_65.7</strong> (None)</dd>

<dd><strong>NODE_155_length_6812_cov_97.4</strong> (None)</dd>

<dd><strong>NODE_156_length_6802_cov_88.3</strong> (None)</dd>

<dd><strong>NODE_157_length_6801_cov_75.3</strong> (None)</dd>

<dd><strong>NODE_158_length_6791_cov_82.9</strong> (None)</dd>

<dd><strong>NODE_159_length_6787_cov_54.3</strong> (None)</dd>

<dd><strong>NODE_160_length_6786_cov_86.8</strong> (None)</dd>

<dd><strong>NODE_161_length_6785_cov_106.0</strong> (None)</dd>

<dd><strong>NODE_162_length_6764_cov_95.6</strong> (None)</dd>

<dd><strong>NODE_163_length_6756_cov_109.0</strong> (None)</dd>

<dd><strong>NODE_164_length_6751_cov_104.6</strong> (None)</dd>

<dd><strong>NODE_165_length_6712_cov_94.0</strong> (None)</dd>

<dd><strong>NODE_166_length_6691_cov_55.4</strong> (None)</dd>

<dd><strong>NODE_167_length_6689_cov_88.4</strong> (None)</dd>

<dd><strong>NODE_168_length_6677_cov_83.1</strong> (None)</dd>

<dd><strong>NODE_169_length_6653_cov_88.3</strong> (None)</dd>

<dd><strong>NODE_170_length_6652_cov_101.9</strong> (None)</dd>

<dd><strong>NODE_171_length_6646_cov_108.0</strong> (None)</dd>

<dd><strong>NODE_172_length_6621_cov_86.0</strong> (None)</dd>

<dd><strong>NODE_173_length_6575_cov_81.2</strong> (None)</dd>

<dd><strong>NODE_174_length_6565_cov_75.3</strong> (None)</dd>

<dd><strong>NODE_175_length_6561_cov_89.0</strong> (None)</dd>

<dd><strong>NODE_176_length_6546_cov_67.7</strong> (None)</dd>

<dd><strong>NODE_177_length_6546_cov_79.6</strong> (None)</dd>

<dd><strong>NODE_178_length_6545_cov_85.3</strong> (None)</dd>

<dd><strong>NODE_179_length_6534_cov_77.1</strong> (None)</dd>

<dd><strong>NODE_180_length_6526_cov_84.9</strong> (None)</dd>

<dd><strong>NODE_181_length_6513_cov_86.3</strong> (None)</dd>

<dd><strong>NODE_182_length_6496_cov_103.5</strong> (None)</dd>

<dd><strong>NODE_183_length_6484_cov_62.0</strong> (None)</dd>

<dd><strong>NODE_184_length_6480_cov_102.3</strong> (None)</dd>

<dd><strong>NODE_185_length_6445_cov_88.1</strong> (None)</dd>

<dd><strong>NODE_186_length_6445_cov_62.7</strong> (None)</dd>

<dd><strong>NODE_187_length_6436_cov_81.0</strong> (None)</dd>

<dd><strong>NODE_188_length_6390_cov_83.1</strong> (None)</dd>

<dd><strong>NODE_189_length_6387_cov_83.8</strong> (None)</dd>

<dd><strong>NODE_190_length_6380_cov_54.6</strong> (None)</dd>

<dd><strong>NODE_191_length_6373_cov_82.5</strong> (None)</dd>

<dd><strong>NODE_192_length_6371_cov_69.8</strong> (None)</dd>

<dd><strong>NODE_193_length_6323_cov_113.1</strong> (None)</dd>

<dd><strong>NODE_194_length_6317_cov_101.7</strong> (None)</dd>

<dd><strong>NODE_195_length_6281_cov_94.4</strong> (None)</dd>

<dd><strong>NODE_196_length_6257_cov_79.0</strong> (None)</dd>

<dd><strong>NODE_197_length_6238_cov_83.7</strong> (None)</dd>

<dd><strong>NODE_198_length_6232_cov_64.9</strong> (None)</dd>

<dd><strong>NODE_199_length_6231_cov_76.9</strong> (None)</dd>

<dd><strong>NODE_200_length_6224_cov_93.8</strong> (None)</dd>

<dd><strong>NODE_201_length_6222_cov_72.7</strong> (None)</dd>

<dd><strong>NODE_202_length_6221_cov_66.2</strong> (None)</dd>

<dd><strong>NODE_203_length_6219_cov_78.9</strong> (None)</dd>

<dd><strong>NODE_204_length_6216_cov_75.6</strong> (None)</dd>

<dd><strong>NODE_205_length_6214_cov_102.9</strong> (None)</dd>

<dd><strong>NODE_206_length_6210_cov_102.8</strong> (None)</dd>

<dd><strong>NODE_207_length_6207_cov_91.4</strong> (None)</dd>

<dd><strong>NODE_208_length_6205_cov_84.7</strong> (None)</dd>

<dd><strong>NODE_209_length_6161_cov_55.3</strong> (None)</dd>

<dd><strong>NODE_210_length_6139_cov_63.7</strong> (None)</dd>

<dd><strong>NODE_211_length_6135_cov_76.1</strong> (None)</dd>

<dd><strong>NODE_212_length_6128_cov_88.8</strong> (None)</dd>

<dd><strong>NODE_213_length_6117_cov_106.9</strong> (None)</dd>

<dd><strong>NODE_214_length_6101_cov_98.5</strong> (None)</dd>

<dd><strong>NODE_215_length_6084_cov_56.0</strong> (None)</dd>

<dd><strong>NODE_216_length_6076_cov_51.5</strong> (None)</dd>

<dd><strong>NODE_217_length_6037_cov_114.7</strong> (None)</dd>

<dd><strong>NODE_218_length_6027_cov_86.2</strong> (None)</dd>

<dd><strong>NODE_219_length_6016_cov_110.9</strong> (None)</dd>

<dd><strong>NODE_220_length_6004_cov_77.9</strong> (None)</dd>

<dd><strong>NODE_221_length_5997_cov_77.4</strong> (None)</dd>

<dd><strong>NODE_222_length_5986_cov_59.6</strong> (None)</dd>

<dd><strong>NODE_223_length_5977_cov_85.2</strong> (None)</dd>

<dd><strong>NODE_224_length_5968_cov_84.1</strong> (None)</dd>

<dd><strong>NODE_225_length_5956_cov_102.0</strong> (None)</dd>

<dd><strong>NODE_226_length_5953_cov_110.5</strong> (None)</dd>

<dd><strong>NODE_227_length_5944_cov_95.2</strong> (None)</dd>

<dd><strong>NODE_228_length_5942_cov_79.9</strong> (None)</dd>

<dd><strong>NODE_229_length_5936_cov_53.7</strong> (None)</dd>

<dd><strong>NODE_230_length_5928_cov_91.8</strong> (None)</dd>

<dd><strong>NODE_231_length_5915_cov_73.3</strong> (None)</dd>

<dd><strong>NODE_232_length_5903_cov_86.2</strong> (None)</dd>

<dd><strong>NODE_233_length_5889_cov_78.6</strong> (None)</dd>

<dd><strong>NODE_234_length_5883_cov_84.6</strong> (None)</dd>

<dd><strong>NODE_235_length_5873_cov_73.7</strong> (None)</dd>

<dd><strong>NODE_236_length_5855_cov_84.3</strong> (None)</dd>

<dd><strong>NODE_237_length_5852_cov_83.1</strong> (None)</dd>

<dd><strong>NODE_238_length_5845_cov_89.9</strong> (None)</dd>

<dd><strong>NODE_239_length_5845_cov_65.6</strong> (None)</dd>

<dd><strong>NODE_240_length_5832_cov_95.7</strong> (None)</dd>

<dd><strong>NODE_241_length_5804_cov_83.9</strong> (None)</dd>

<dd><strong>NODE_242_length_5795_cov_74.1</strong> (None)</dd>

<dd><strong>NODE_243_length_5783_cov_85.8</strong> (None)</dd>

<dd><strong>NODE_244_length_5777_cov_91.4</strong> (None)</dd>

<dd><strong>NODE_245_length_5771_cov_96.0</strong> (None)</dd>

<dd><strong>NODE_246_length_5759_cov_86.8</strong> (None)</dd>

<dd><strong>NODE_247_length_5746_cov_77.2</strong> (None)</dd>

<dd><strong>NODE_248_length_5739_cov_84.1</strong> (None)</dd>

<dd><strong>NODE_249_length_5739_cov_87.4</strong> (None)</dd>

<dd><strong>NODE_250_length_5739_cov_82.3</strong> (None)</dd>

<dd><strong>NODE_251_length_5725_cov_88.0</strong> (None)</dd>

<dd><strong>NODE_252_length_5715_cov_61.3</strong> (None)</dd>

<dd><strong>NODE_253_length_5702_cov_66.5</strong> (None)</dd>

<dd><strong>NODE_254_length_5700_cov_63.4</strong> (None)</dd>

<dd><strong>NODE_255_length_5686_cov_103.9</strong> (None)</dd>

<dd><strong>NODE_256_length_5682_cov_64.3</strong> (None)</dd>

<dd><strong>NODE_257_length_5668_cov_83.6</strong> (None)</dd>

<dd><strong>NODE_258_length_5666_cov_82.7</strong> (None)</dd>

<dd><strong>NODE_260_length_5652_cov_125.1</strong> (None)</dd>

<dd><strong>NODE_261_length_5647_cov_90.2</strong> (None)</dd>

<dd><strong>NODE_262_length_5625_cov_98.6</strong> (None)</dd>

<dd><strong>NODE_263_length_5623_cov_64.6</strong> (None)</dd>

<dd><strong>NODE_264_length_5621_cov_71.9</strong> (None)</dd>

<dd><strong>NODE_265_length_5618_cov_62.9</strong> (None)</dd>

<dd><strong>NODE_266_length_5615_cov_52.7</strong> (None)</dd>

<dd><strong>NODE_267_length_5614_cov_83.5</strong> (None)</dd>

<dd><strong>NODE_268_length_5608_cov_53.5</strong> (None)</dd>

<dd><strong>NODE_269_length_5593_cov_101.7</strong> (None)</dd>

<dd><strong>NODE_270_length_5579_cov_81.0</strong> (None)</dd>

<dd><strong>NODE_271_length_5577_cov_72.2</strong> (None)</dd>

<dd><strong>NODE_272_length_5575_cov_61.8</strong> (None)</dd>

<dd><strong>NODE_273_length_5562_cov_93.5</strong> (None)</dd>

<dd><strong>NODE_274_length_5561_cov_80.8</strong> (None)</dd>

<dd><strong>NODE_275_length_5553_cov_86.7</strong> (None)</dd>

<dd><strong>NODE_276_length_5531_cov_77.7</strong> (None)</dd>

<dd><strong>NODE_277_length_5518_cov_89.7</strong> (None)</dd>

<dd><strong>NODE_278_length_5507_cov_75.8</strong> (None)</dd>

<dd><strong>NODE_279_length_5463_cov_60.7</strong> (None)</dd>

<dd><strong>NODE_280_length_5461_cov_80.1</strong> (None)</dd>

<dd><strong>NODE_281_length_5458_cov_86.9</strong> (None)</dd>

<dd><strong>NODE_282_length_5441_cov_79.6</strong> (None)</dd>

<dd><strong>NODE_283_length_5438_cov_54.1</strong> (None)</dd>

<dd><strong>NODE_284_length_5431_cov_69.1</strong> (None)</dd>

<dd><strong>NODE_285_length_5427_cov_64.1</strong> (None)</dd>

<dd><strong>NODE_286_length_5418_cov_95.2</strong> (None)</dd>

<dd><strong>NODE_287_length_5418_cov_106.8</strong> (None)</dd>

<dd><strong>NODE_288_length_5401_cov_63.9</strong> (None)</dd>

<dd><strong>NODE_289_length_5399_cov_66.9</strong> (None)</dd>

<dd><strong>NODE_290_length_5387_cov_59.6</strong> (None)</dd>

<dd><strong>NODE_291_length_5379_cov_80.7</strong> (None)</dd>

<dd><strong>NODE_292_length_5375_cov_92.1</strong> (None)</dd>

<dd><strong>NODE_293_length_5367_cov_89.9</strong> (None)</dd>

<dd><strong>NODE_294_length_5366_cov_78.8</strong> (None)</dd>

<dd><strong>NODE_295_length_5365_cov_83.0</strong> (None)</dd>

<dd><strong>NODE_296_length_5347_cov_66.7</strong> (None)</dd>

<dd><strong>NODE_297_length_5346_cov_91.6</strong> (None)</dd>

<dd><strong>NODE_298_length_5344_cov_96.3</strong> (None)</dd>

<dd><strong>NODE_299_length_5342_cov_63.0</strong> (None)</dd>

<dd><strong>NODE_300_length_5337_cov_52.6</strong> (None)</dd>

<dd><strong>NODE_301_length_5335_cov_108.0</strong> (None)</dd>

<dd><strong>NODE_302_length_5323_cov_92.0</strong> (None)</dd>

<dd><strong>NODE_303_length_5300_cov_81.6</strong> (None)</dd>

<dd><strong>NODE_304_length_5281_cov_90.8</strong> (None)</dd>

<dd><strong>NODE_305_length_5281_cov_71.9</strong> (None)</dd>

<dd><strong>NODE_306_length_5270_cov_71.8</strong> (None)</dd>

<dd><strong>NODE_307_length_5268_cov_56.5</strong> (None)</dd>

<dd><strong>NODE_308_length_5268_cov_54.2</strong> (None)</dd>

<dd><strong>NODE_309_length_5267_cov_95.5</strong> (None)</dd>

<dd><strong>NODE_310_length_5258_cov_75.2</strong> (None)</dd>

<dd><strong>NODE_311_length_5255_cov_91.5</strong> (None)</dd>

<dd><strong>NODE_312_length_5251_cov_96.9</strong> (None)</dd>

<dd><strong>NODE_313_length_5229_cov_81.7</strong> (None)</dd>

<dd><strong>NODE_314_length_5228_cov_86.3</strong> (None)</dd>

<dd><strong>NODE_315_length_5217_cov_93.2</strong> (None)</dd>

<dd><strong>NODE_316_length_5206_cov_62.9</strong> (None)</dd>

<dd><strong>NODE_317_length_5201_cov_93.9</strong> (None)</dd>

<dd><strong>NODE_318_length_5181_cov_86.2</strong> (None)</dd>

<dd><strong>NODE_319_length_5181_cov_81.6</strong> (None)</dd>

<dd><strong>NODE_320_length_5172_cov_73.9</strong> (None)</dd>

<dd><strong>NODE_321_length_5165_cov_77.4</strong> (None)</dd>

<dd><strong>NODE_322_length_5162_cov_98.6</strong> (None)</dd>

<dd><strong>NODE_323_length_5155_cov_64.5</strong> (None)</dd>

<dd><strong>NODE_324_length_5154_cov_86.5</strong> (None)</dd>

<dd><strong>NODE_325_length_5152_cov_78.9</strong> (None)</dd>

<dd><strong>NODE_326_length_5144_cov_78.2</strong> (None)</dd>

<dd><strong>NODE_327_length_5136_cov_89.1</strong> (None)</dd>

<dd><strong>NODE_328_length_5133_cov_111.3</strong> (None)</dd>

<dd><strong>NODE_329_length_5128_cov_76.3</strong> (None)</dd>

<dd><strong>NODE_330_length_5123_cov_53.1</strong> (None)</dd>

<dd><strong>NODE_331_length_5114_cov_76.3</strong> (None)</dd>

<dd><strong>NODE_332_length_5114_cov_85.2</strong> (None)</dd>

<dd><strong>NODE_333_length_5111_cov_68.3</strong> (None)</dd>

<dd><strong>NODE_334_length_5101_cov_92.9</strong> (None)</dd>

<dd><strong>NODE_335_length_5091_cov_82.3</strong> (None)</dd>

<dd><strong>NODE_336_length_5065_cov_57.8</strong> (None)</dd>

<dd><strong>NODE_337_length_5058_cov_97.9</strong> (None)</dd>

<dd><strong>NODE_339_length_5055_cov_82.1</strong> (None)</dd>

<dd><strong>NODE_340_length_5052_cov_116.0</strong> (None)</dd>

<dd><strong>NODE_341_length_5042_cov_91.6</strong> (None)</dd>

<dd><strong>NODE_342_length_5038_cov_76.5</strong> (None)</dd>

<dd><strong>NODE_343_length_5032_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_344_length_5029_cov_59.4</strong> (None)</dd>

<dd><strong>NODE_345_length_5016_cov_85.7</strong> (None)</dd>

<dd><strong>NODE_346_length_5001_cov_102.2</strong> (None)</dd>

<dd><strong>NODE_347_length_4996_cov_65.7</strong> (None)</dd>

<dd><strong>NODE_348_length_4990_cov_93.8</strong> (None)</dd>

<dd><strong>NODE_349_length_4990_cov_95.8</strong> (None)</dd>

<dd><strong>NODE_350_length_4986_cov_107.2</strong> (None)</dd>

<dd><strong>NODE_351_length_4973_cov_85.0</strong> (None)</dd>

<dd><strong>NODE_352_length_4961_cov_88.0</strong> (None)</dd>

<dd><strong>NODE_353_length_4958_cov_107.2</strong> (None)</dd>

<dd><strong>NODE_354_length_4947_cov_65.0</strong> (None)</dd>

<dd><strong>NODE_355_length_4946_cov_52.5</strong> (None)</dd>

<dd><strong>NODE_356_length_4946_cov_97.2</strong> (None)</dd>

<dd><strong>NODE_357_length_4939_cov_73.8</strong> (None)</dd>

<dd><strong>NODE_358_length_4926_cov_78.1</strong> (None)</dd>

<dd><strong>NODE_359_length_4925_cov_91.0</strong> (None)</dd>

<dd><strong>NODE_360_length_4914_cov_70.0</strong> (None)</dd>

<dd><strong>NODE_361_length_4903_cov_79.8</strong> (None)</dd>

<dd><strong>NODE_362_length_4902_cov_96.7</strong> (None)</dd>

<dd><strong>NODE_363_length_4897_cov_75.5</strong> (None)</dd>

<dd><strong>NODE_364_length_4886_cov_70.7</strong> (None)</dd>

<dd><strong>NODE_365_length_4882_cov_100.5</strong> (None)</dd>

<dd><strong>NODE_366_length_4878_cov_64.5</strong> (None)</dd>

<dd><strong>NODE_367_length_4873_cov_53.8</strong> (None)</dd>

<dd><strong>NODE_368_length_4870_cov_98.6</strong> (None)</dd>

<dd><strong>NODE_369_length_4868_cov_98.2</strong> (None)</dd>

<dd><strong>NODE_370_length_4861_cov_57.6</strong> (None)</dd>

<dd><strong>NODE_371_length_4829_cov_101.0</strong> (None)</dd>

<dd><strong>NODE_372_length_4814_cov_54.6</strong> (None)</dd>

<dd><strong>NODE_373_length_4809_cov_84.1</strong> (None)</dd>

<dd><strong>NODE_374_length_4808_cov_61.0</strong> (None)</dd>

<dd><strong>NODE_375_length_4800_cov_53.3</strong> (None)</dd>

<dd><strong>NODE_376_length_4797_cov_81.2</strong> (None)</dd>

<dd><strong>NODE_377_length_4795_cov_68.9</strong> (None)</dd>

<dd><strong>NODE_378_length_4793_cov_100.7</strong> (None)</dd>

<dd><strong>NODE_379_length_4792_cov_86.9</strong> (None)</dd>

<dd><strong>NODE_380_length_4788_cov_58.5</strong> (None)</dd>

<dd><strong>NODE_382_length_4774_cov_68.8</strong> (None)</dd>

<dd><strong>NODE_383_length_4774_cov_77.9</strong> (None)</dd>

<dd><strong>NODE_384_length_4773_cov_56.7</strong> (None)</dd>

<dd><strong>NODE_385_length_4770_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_386_length_4765_cov_52.9</strong> (None)</dd>

<dd><strong>NODE_387_length_4763_cov_56.9</strong> (None)</dd>

<dd><strong>NODE_388_length_4755_cov_84.6</strong> (None)</dd>

<dd><strong>NODE_389_length_4753_cov_54.4</strong> (None)</dd>

<dd><strong>NODE_390_length_4748_cov_77.6</strong> (None)</dd>

<dd><strong>NODE_391_length_4744_cov_80.4</strong> (None)</dd>

<dd><strong>NODE_392_length_4741_cov_99.5</strong> (None)</dd>

<dd><strong>NODE_393_length_4738_cov_61.6</strong> (None)</dd>

<dd><strong>NODE_394_length_4736_cov_51.7</strong> (None)</dd>

<dd><strong>NODE_395_length_4727_cov_87.2</strong> (None)</dd>

<dd><strong>NODE_396_length_4726_cov_60.7</strong> (None)</dd>

<dd><strong>NODE_397_length_4725_cov_73.4</strong> (None)</dd>

<dd><strong>NODE_398_length_4724_cov_97.2</strong> (None)</dd>

<dd><strong>NODE_399_length_4721_cov_77.0</strong> (None)</dd>

<dd><strong>NODE_400_length_4719_cov_77.0</strong> (None)</dd>

<dd><strong>NODE_401_length_4709_cov_99.2</strong> (None)</dd>

<dd><strong>NODE_402_length_4705_cov_79.3</strong> (None)</dd>

<dd><strong>NODE_403_length_4701_cov_85.5</strong> (None)</dd>

<dd><strong>NODE_404_length_4691_cov_102.3</strong> (None)</dd>

<dd><strong>NODE_405_length_4690_cov_100.5</strong> (None)</dd>

<dd><strong>NODE_406_length_4689_cov_52.8</strong> (None)</dd>

<dd><strong>NODE_407_length_4682_cov_69.0</strong> (None)</dd>

<dd><strong>NODE_408_length_4680_cov_114.0</strong> (None)</dd>

<dd><strong>NODE_409_length_4679_cov_69.2</strong> (None)</dd>

<dd><strong>NODE_410_length_4679_cov_80.8</strong> (None)</dd>

<dd><strong>NODE_411_length_4676_cov_89.6</strong> (None)</dd>

<dd><strong>NODE_412_length_4661_cov_61.4</strong> (None)</dd>

<dd><strong>NODE_413_length_4661_cov_63.0</strong> (None)</dd>

<dd><strong>NODE_414_length_4656_cov_88.9</strong> (None)</dd>

<dd><strong>NODE_415_length_4654_cov_54.5</strong> (None)</dd>

<dd><strong>NODE_416_length_4648_cov_62.3</strong> (None)</dd>

<dd><strong>NODE_417_length_4645_cov_65.4</strong> (None)</dd>

<dd><strong>NODE_418_length_4643_cov_69.4</strong> (None)</dd>

<dd><strong>NODE_419_length_4641_cov_85.1</strong> (None)</dd>

<dd><strong>NODE_420_length_4639_cov_90.2</strong> (None)</dd>

<dd><strong>NODE_421_length_4638_cov_76.0</strong> (None)</dd>

<dd><strong>NODE_422_length_4628_cov_99.6</strong> (None)</dd>

<dd><strong>NODE_423_length_4623_cov_56.6</strong> (None)</dd>

<dd><strong>NODE_424_length_4615_cov_56.2</strong> (None)</dd>

<dd><strong>NODE_425_length_4607_cov_79.2</strong> (None)</dd>

<dd><strong>NODE_426_length_4607_cov_79.8</strong> (None)</dd>

<dd><strong>NODE_427_length_4601_cov_59.8</strong> (None)</dd>

<dd><strong>NODE_428_length_4598_cov_87.9</strong> (None)</dd>

<dd><strong>NODE_429_length_4594_cov_78.1</strong> (None)</dd>

<dd><strong>NODE_430_length_4591_cov_62.2</strong> (None)</dd>

<dd><strong>NODE_431_length_4589_cov_52.5</strong> (None)</dd>

<dd><strong>NODE_432_length_4574_cov_65.2</strong> (None)</dd>

<dd><strong>NODE_433_length_4570_cov_53.5</strong> (None)</dd>

<dd><strong>NODE_434_length_4566_cov_93.7</strong> (None)</dd>

<dd><strong>NODE_435_length_4562_cov_58.3</strong> (None)</dd>

<dd><strong>NODE_436_length_4559_cov_72.0</strong> (None)</dd>

<dd><strong>NODE_437_length_4558_cov_56.6</strong> (None)</dd>

<dd><strong>NODE_438_length_4557_cov_56.2</strong> (None)</dd>

<dd><strong>NODE_439_length_4554_cov_82.9</strong> (None)</dd>

<dd><strong>NODE_440_length_4552_cov_60.8</strong> (None)</dd>

<dd><strong>NODE_441_length_4551_cov_63.5</strong> (None)</dd>

<dd><strong>NODE_442_length_4550_cov_100.5</strong> (None)</dd>

<dd><strong>NODE_443_length_4549_cov_105.4</strong> (None)</dd>

<dd><strong>NODE_444_length_4538_cov_61.1</strong> (None)</dd>

<dd><strong>NODE_445_length_4533_cov_100.9</strong> (None)</dd>

<dd><strong>NODE_446_length_4532_cov_79.3</strong> (None)</dd>

<dd><strong>NODE_447_length_4528_cov_57.4</strong> (None)</dd>

<dd><strong>NODE_448_length_4527_cov_91.4</strong> (None)</dd>

<dd><strong>NODE_449_length_4526_cov_78.1</strong> (None)</dd>

<dd><strong>NODE_450_length_4524_cov_96.0</strong> (None)</dd>

<dd><strong>NODE_451_length_4522_cov_62.5</strong> (None)</dd>

<dd><strong>NODE_452_length_4521_cov_64.0</strong> (None)</dd>

<dd><strong>NODE_453_length_4516_cov_51.6</strong> (None)</dd>

<dd><strong>NODE_454_length_4515_cov_57.2</strong> (None)</dd>

<dd><strong>NODE_455_length_4514_cov_83.6</strong> (None)</dd>

<dd><strong>NODE_456_length_4513_cov_87.9</strong> (None)</dd>

<dd><strong>NODE_457_length_4508_cov_80.0</strong> (None)</dd>

<dd><strong>NODE_458_length_4501_cov_51.4</strong> (None)</dd>

<dd><strong>NODE_459_length_4499_cov_77.6</strong> (None)</dd>

<dd><strong>NODE_460_length_4490_cov_90.1</strong> (None)</dd>

<dd><strong>NODE_461_length_4489_cov_62.5</strong> (None)</dd>

<dd><strong>NODE_462_length_4489_cov_82.1</strong> (None)</dd>

<dd><strong>NODE_463_length_4479_cov_53.1</strong> (None)</dd>

<dd><strong>NODE_464_length_4474_cov_71.2</strong> (None)</dd>

<dd><strong>NODE_465_length_4469_cov_51.8</strong> (None)</dd>

<dd><strong>NODE_466_length_4464_cov_74.1</strong> (None)</dd>

<dd><strong>NODE_467_length_4463_cov_78.5</strong> (None)</dd>

<dd><strong>NODE_468_length_4462_cov_59.1</strong> (None)</dd>

<dd><strong>NODE_469_length_4461_cov_96.1</strong> (None)</dd>

<dd><strong>NODE_470_length_4459_cov_67.9</strong> (None)</dd>

<dd><strong>NODE_471_length_4457_cov_64.4</strong> (None)</dd>

<dd><strong>NODE_472_length_4457_cov_66.4</strong> (None)</dd>

<dd><strong>NODE_473_length_4448_cov_53.5</strong> (None)</dd>

<dd><strong>NODE_474_length_4447_cov_77.7</strong> (None)</dd>

<dd><strong>NODE_475_length_4443_cov_57.9</strong> (None)</dd>

<dd><strong>NODE_476_length_4443_cov_95.2</strong> (None)</dd>

<dd><strong>NODE_477_length_4434_cov_83.8</strong> (None)</dd>

<dd><strong>NODE_478_length_4429_cov_97.8</strong> (None)</dd>

<dd><strong>NODE_479_length_4422_cov_50.4</strong> (None)</dd>

<dd><strong>NODE_480_length_4422_cov_98.5</strong> (None)</dd>

<dd><strong>NODE_481_length_4417_cov_97.2</strong> (None)</dd>

<dd><strong>NODE_482_length_4416_cov_81.7</strong> (None)</dd>

<dd><strong>NODE_483_length_4416_cov_81.9</strong> (None)</dd>

<dd><strong>NODE_484_length_4416_cov_52.4</strong> (None)</dd>

<dd><strong>NODE_485_length_4414_cov_105.5</strong> (None)</dd>

<dd><strong>NODE_486_length_4409_cov_83.5</strong> (None)</dd>

<dd><strong>NODE_487_length_4399_cov_95.5</strong> (None)</dd>

<dd><strong>NODE_488_length_4390_cov_75.1</strong> (None)</dd>

<dd><strong>NODE_489_length_4384_cov_62.6</strong> (None)</dd>

<dd><strong>NODE_490_length_4376_cov_75.0</strong> (None)</dd>

<dd><strong>NODE_491_length_4374_cov_60.2</strong> (None)</dd>

<dd><strong>NODE_492_length_4365_cov_78.2</strong> (None)</dd>

<dd><strong>NODE_493_length_4365_cov_174.4</strong> (None)</dd>

<dd><strong>NODE_494_length_4364_cov_52.7</strong> (None)</dd>

<dd><strong>NODE_495_length_4362_cov_105.9</strong> (None)</dd>

<dd><strong>NODE_496_length_4351_cov_99.3</strong> (None)</dd>

<dd><strong>NODE_497_length_4343_cov_86.5</strong> (None)</dd>

<dd><strong>NODE_498_length_4340_cov_74.1</strong> (None)</dd>

<dd><strong>NODE_499_length_4338_cov_101.0</strong> (None)</dd>

<dd><strong>NODE_500_length_4335_cov_60.8</strong> (None)</dd>

<dd><strong>NODE_501_length_4328_cov_75.7</strong> (None)</dd>

<dd><strong>NODE_502_length_4328_cov_85.4</strong> (None)</dd>

<dd><strong>NODE_503_length_4326_cov_61.3</strong> (None)</dd>

<dd><strong>NODE_504_length_4326_cov_54.6</strong> (None)</dd>

<dd><strong>NODE_505_length_4323_cov_64.4</strong> (None)</dd>

<dd><strong>NODE_506_length_4318_cov_54.3</strong> (None)</dd>

<dd><strong>NODE_507_length_4317_cov_82.2</strong> (None)</dd>

<dd><strong>NODE_508_length_4317_cov_54.5</strong> (None)</dd>

<dd><strong>NODE_509_length_4316_cov_57.2</strong> (None)</dd>

<dd><strong>NODE_510_length_4315_cov_76.1</strong> (None)</dd>

<dd><strong>NODE_511_length_4309_cov_49.9</strong> (None)</dd>

<dd><strong>NODE_512_length_4307_cov_72.3</strong> (None)</dd>

<dd><strong>NODE_513_length_4303_cov_75.1</strong> (None)</dd>

<dd><strong>NODE_514_length_4303_cov_51.4</strong> (None)</dd>

<dd><strong>NODE_515_length_4302_cov_86.0</strong> (None)</dd>

<dd><strong>NODE_516_length_4302_cov_53.3</strong> (None)</dd>

<dd><strong>NODE_517_length_4301_cov_61.3</strong> (None)</dd>

<dd><strong>NODE_518_length_4300_cov_62.5</strong> (None)</dd>

<dd><strong>NODE_519_length_4296_cov_105.3</strong> (None)</dd>

<dd><strong>NODE_520_length_4293_cov_70.8</strong> (None)</dd>

<dd><strong>NODE_521_length_4293_cov_54.0</strong> (None)</dd>

<dd><strong>NODE_522_length_4293_cov_64.0</strong> (None)</dd>

<dd><strong>NODE_523_length_4291_cov_53.0</strong> (None)</dd>

<dd><strong>NODE_524_length_4291_cov_83.3</strong> (None)</dd>

<dd><strong>NODE_525_length_4285_cov_84.6</strong> (None)</dd>

<dd><strong>NODE_526_length_4278_cov_98.7</strong> (None)</dd>

<dd><strong>NODE_527_length_4270_cov_65.9</strong> (None)</dd>

<dd><strong>NODE_528_length_4258_cov_84.6</strong> (None)</dd>

<dd><strong>NODE_529_length_4255_cov_76.5</strong> (None)</dd>

<dd><strong>NODE_530_length_4255_cov_84.4</strong> (None)</dd>

<dd><strong>NODE_531_length_4246_cov_68.5</strong> (None)</dd>

<dd><strong>NODE_532_length_4245_cov_61.1</strong> (None)</dd>

<dd><strong>NODE_533_length_4243_cov_84.9</strong> (None)</dd>

<dd><strong>NODE_534_length_4238_cov_68.1</strong> (None)</dd>

<dd><strong>NODE_535_length_4229_cov_56.1</strong> (None)</dd>

<dd><strong>NODE_536_length_4227_cov_110.6</strong> (None)</dd>

<dd><strong>NODE_537_length_4225_cov_83.7</strong> (None)</dd>

<dd><strong>NODE_538_length_4225_cov_91.9</strong> (None)</dd>

<dd><strong>NODE_539_length_4221_cov_86.6</strong> (None)</dd>

<dd><strong>NODE_540_length_4217_cov_80.9</strong> (None)</dd>

<dd><strong>NODE_541_length_4216_cov_78.5</strong> (None)</dd>

<dd><strong>NODE_542_length_4209_cov_54.1</strong> (None)</dd>

<dd><strong>NODE_543_length_4208_cov_55.2</strong> (None)</dd>

<dd><strong>NODE_544_length_4206_cov_105.1</strong> (None)</dd>

<dd><strong>NODE_545_length_4199_cov_56.6</strong> (None)</dd>

<dd><strong>NODE_546_length_4199_cov_101.0</strong> (None)</dd>

<dd><strong>NODE_547_length_4192_cov_101.5</strong> (None)</dd>

<dd><strong>NODE_548_length_4190_cov_120.1</strong> (None)</dd>

<dd><strong>NODE_549_length_4187_cov_65.8</strong> (None)</dd>

<dd><strong>NODE_550_length_4186_cov_59.0</strong> (None)</dd>

<dd><strong>NODE_551_length_4185_cov_106.9</strong> (None)</dd>

<dd><strong>NODE_552_length_4185_cov_61.9</strong> (None)</dd>

<dd><strong>NODE_553_length_4177_cov_84.4</strong> (None)</dd>

<dd><strong>NODE_554_length_4169_cov_83.1</strong> (None)</dd>

<dd><strong>NODE_555_length_4162_cov_77.8</strong> (None)</dd>

<dd><strong>NODE_556_length_4157_cov_103.0</strong> (None)</dd>

<dd><strong>NODE_557_length_4150_cov_87.0</strong> (None)</dd>

<dd><strong>NODE_558_length_4148_cov_89.6</strong> (None)</dd>

<dd><strong>NODE_559_length_4144_cov_84.9</strong> (None)</dd>

<dd><strong>NODE_560_length_4143_cov_69.5</strong> (None)</dd>

<dd><strong>NODE_561_length_4140_cov_55.0</strong> (None)</dd>

<dd><strong>NODE_562_length_4138_cov_52.0</strong> (None)</dd>

<dd><strong>NODE_563_length_4133_cov_55.1</strong> (None)</dd>

<dd><strong>NODE_564_length_4132_cov_86.0</strong> (None)</dd>

<dd><strong>NODE_565_length_4131_cov_76.0</strong> (None)</dd>

<dd><strong>NODE_566_length_4130_cov_61.5</strong> (None)</dd>

<dd><strong>NODE_567_length_4128_cov_57.7</strong> (None)</dd>

<dd><strong>NODE_568_length_4123_cov_80.3</strong> (None)</dd>

<dd><strong>NODE_569_length_4122_cov_56.9</strong> (None)</dd>

<dd><strong>NODE_570_length_4120_cov_58.1</strong> (None)</dd>

<dd><strong>NODE_571_length_4108_cov_82.4</strong> (None)</dd>

<dd><strong>NODE_572_length_4097_cov_66.3</strong> (None)</dd>

<dd><strong>NODE_573_length_4092_cov_95.7</strong> (None)</dd>

<dd><strong>NODE_574_length_4091_cov_105.7</strong> (None)</dd>

<dd><strong>NODE_575_length_4087_cov_52.9</strong> (None)</dd>

<dd><strong>NODE_576_length_4078_cov_105.2</strong> (None)</dd>

<dd><strong>NODE_577_length_4076_cov_72.0</strong> (None)</dd>

<dd><strong>NODE_578_length_4076_cov_56.4</strong> (None)</dd>

<dd><strong>NODE_579_length_4074_cov_86.2</strong> (None)</dd>

<dd><strong>NODE_580_length_4068_cov_101.8</strong> (None)</dd>

<dd><strong>NODE_581_length_4064_cov_60.7</strong> (None)</dd>

<dd><strong>NODE_582_length_4063_cov_94.2</strong> (None)</dd>

<dd><strong>NODE_583_length_4062_cov_87.0</strong> (None)</dd>

<dd><strong>NODE_584_length_4062_cov_78.9</strong> (None)</dd>

<dd><strong>NODE_585_length_4062_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_586_length_4059_cov_62.7</strong> (None)</dd>

<dd><strong>NODE_587_length_4055_cov_91.3</strong> (None)</dd>

<dd><strong>NODE_588_length_4054_cov_92.8</strong> (None)</dd>

<dd><strong>NODE_589_length_4053_cov_110.7</strong> (None)</dd>

<dd><strong>NODE_590_length_4046_cov_93.2</strong> (None)</dd>

<dd><strong>NODE_591_length_4044_cov_75.8</strong> (None)</dd>

<dd><strong>NODE_592_length_4039_cov_86.4</strong> (None)</dd>

<dd><strong>NODE_593_length_4034_cov_60.0</strong> (None)</dd>

<dd><strong>NODE_594_length_4033_cov_114.3</strong> (None)</dd>

<dd><strong>NODE_595_length_4031_cov_77.1</strong> (None)</dd>

<dd><strong>NODE_596_length_4030_cov_107.8</strong> (None)</dd>

<dd><strong>NODE_597_length_4029_cov_52.7</strong> (None)</dd>

<dd><strong>NODE_598_length_4029_cov_63.3</strong> (None)</dd>

<dd><strong>NODE_599_length_4026_cov_91.5</strong> (None)</dd>

<dd><strong>NODE_600_length_4025_cov_91.8</strong> (None)</dd>

<dd><strong>NODE_601_length_4020_cov_70.4</strong> (None)</dd>

<dd><strong>NODE_602_length_4018_cov_62.3</strong> (None)</dd>

<dd><strong>NODE_603_length_4016_cov_68.3</strong> (None)</dd>

<dd><strong>NODE_604_length_4015_cov_53.6</strong> (None)</dd>

<dd><strong>NODE_605_length_4014_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_606_length_4014_cov_93.6</strong> (None)</dd>

<dd><strong>NODE_607_length_4009_cov_57.7</strong> (None)</dd>

<dd><strong>NODE_608_length_4001_cov_64.2</strong> (None)</dd>

<dd><strong>NODE_609_length_3998_cov_49.3</strong> (None)</dd>

<dd><strong>NODE_610_length_3997_cov_66.6</strong> (None)</dd>

<dd><strong>NODE_611_length_3996_cov_64.9</strong> (None)</dd>

<dd><strong>NODE_612_length_3996_cov_111.5</strong> (None)</dd>

<dd><strong>NODE_613_length_3996_cov_83.1</strong> (None)</dd>

<dd><strong>NODE_614_length_3994_cov_97.7</strong> (None)</dd>

<dd><strong>NODE_615_length_3987_cov_53.2</strong> (None)</dd>

<dd><strong>NODE_616_length_3983_cov_59.3</strong> (None)</dd>

<dd><strong>NODE_617_length_3981_cov_82.4</strong> (None)</dd>

<dd><strong>NODE_618_length_3971_cov_54.8</strong> (None)</dd>

<dd><strong>NODE_619_length_3969_cov_62.3</strong> (None)</dd>

<dd><strong>NODE_620_length_3968_cov_100.4</strong> (None)</dd>

<dd><strong>NODE_621_length_3965_cov_74.3</strong> (None)</dd>

<dd><strong>NODE_622_length_3959_cov_106.1</strong> (None)</dd>

<dd><strong>NODE_623_length_3959_cov_65.1</strong> (None)</dd>

<dd><strong>NODE_624_length_3955_cov_52.3</strong> (None)</dd>

<dd><strong>NODE_625_length_3950_cov_55.3</strong> (None)</dd>

<dd><strong>NODE_626_length_3944_cov_119.2</strong> (None)</dd>

<dd><strong>NODE_627_length_3941_cov_96.0</strong> (None)</dd>

<dd><strong>NODE_628_length_3941_cov_99.6</strong> (None)</dd>

<dd><strong>NODE_629_length_3938_cov_50.7</strong> (None)</dd>

<dd><strong>NODE_630_length_3931_cov_71.4</strong> (None)</dd>

<dd><strong>NODE_631_length_3931_cov_118.3</strong> (None)</dd>

<dd><strong>NODE_632_length_3928_cov_61.0</strong> (None)</dd>

<dd><strong>NODE_633_length_3922_cov_61.3</strong> (None)</dd>

<dd><strong>NODE_634_length_3915_cov_59.5</strong> (None)</dd>

<dd><strong>NODE_635_length_3909_cov_66.5</strong> (None)</dd>

<dd><strong>NODE_636_length_3907_cov_99.7</strong> (None)</dd>

<dd><strong>NODE_637_length_3903_cov_51.4</strong> (None)</dd>

<dd><strong>NODE_638_length_3902_cov_47.1</strong> (None)</dd>

<dd><strong>NODE_639_length_3898_cov_62.8</strong> (None)</dd>

<dd><strong>NODE_640_length_3898_cov_66.8</strong> (None)</dd>

<dd><strong>NODE_641_length_3897_cov_83.9</strong> (None)</dd>

<dd><strong>NODE_642_length_3892_cov_81.7</strong> (None)</dd>

<dd><strong>NODE_643_length_3889_cov_52.7</strong> (None)</dd>

<dd><strong>NODE_644_length_3889_cov_71.6</strong> (None)</dd>

<dd><strong>NODE_645_length_3887_cov_101.4</strong> (None)</dd>

<dd><strong>NODE_646_length_3882_cov_58.8</strong> (None)</dd>

<dd><strong>NODE_647_length_3880_cov_55.5</strong> (None)</dd>

<dd><strong>NODE_648_length_3875_cov_70.3</strong> (None)</dd>

<dd><strong>NODE_649_length_3871_cov_83.1</strong> (None)</dd>

<dd><strong>NODE_650_length_3870_cov_69.1</strong> (None)</dd>

<dd><strong>NODE_651_length_3870_cov_77.3</strong> (None)</dd>

<dd><strong>NODE_652_length_3868_cov_62.4</strong> (None)</dd>

<dd><strong>NODE_653_length_3866_cov_65.5</strong> (None)</dd>

<dd><strong>NODE_654_length_3864_cov_88.7</strong> (None)</dd>

<dd><strong>NODE_655_length_3862_cov_86.0</strong> (None)</dd>

<dd><strong>NODE_656_length_3858_cov_55.0</strong> (None)</dd>

<dd><strong>NODE_657_length_3856_cov_91.7</strong> (None)</dd>

<dd><strong>NODE_658_length_3852_cov_62.2</strong> (None)</dd>

<dd><strong>NODE_659_length_3850_cov_99.0</strong> (None)</dd>

<dd><strong>NODE_660_length_3847_cov_63.2</strong> (None)</dd>

<dd><strong>NODE_661_length_3846_cov_55.7</strong> (None)</dd>

<dd><strong>NODE_662_length_3845_cov_83.7</strong> (None)</dd>

<dd><strong>NODE_663_length_3842_cov_89.5</strong> (None)</dd>

<dd><strong>NODE_664_length_3841_cov_69.3</strong> (None)</dd>

<dd><strong>NODE_665_length_3836_cov_54.8</strong> (None)</dd>

<dd><strong>NODE_666_length_3835_cov_65.4</strong> (None)</dd>

<dd><strong>NODE_667_length_3827_cov_52.3</strong> (None)</dd>

<dd><strong>NODE_668_length_3826_cov_68.9</strong> (None)</dd>

<dd><strong>NODE_669_length_3826_cov_62.4</strong> (None)</dd>

<dd><strong>NODE_670_length_3822_cov_107.2</strong> (None)</dd>

<dd><strong>NODE_671_length_3822_cov_81.0</strong> (None)</dd>

<dd><strong>NODE_672_length_3818_cov_48.5</strong> (None)</dd>

<dd><strong>NODE_673_length_3817_cov_80.7</strong> (None)</dd>

<dd><strong>NODE_674_length_3815_cov_49.8</strong> (None)</dd>

<dd><strong>NODE_675_length_3807_cov_53.5</strong> (None)</dd>

<dd><strong>NODE_676_length_3802_cov_59.2</strong> (None)</dd>

<dd><strong>NODE_677_length_3800_cov_49.7</strong> (None)</dd>

<dd><strong>NODE_678_length_3799_cov_82.2</strong> (None)</dd>

<dd><strong>NODE_679_length_3792_cov_56.2</strong> (None)</dd>

<dd><strong>NODE_680_length_3789_cov_66.8</strong> (None)</dd>

<dd><strong>NODE_681_length_3785_cov_113.4</strong> (None)</dd>

<dd><strong>NODE_682_length_3785_cov_52.6</strong> (None)</dd>

<dd><strong>NODE_683_length_3784_cov_84.1</strong> (None)</dd>

<dd><strong>NODE_684_length_3779_cov_94.9</strong> (None)</dd>

<dd><strong>NODE_685_length_3779_cov_77.1</strong> (None)</dd>

<dd><strong>NODE_686_length_3776_cov_74.8</strong> (None)</dd>

<dd><strong>NODE_687_length_3774_cov_58.6</strong> (None)</dd>

<dd><strong>NODE_688_length_3770_cov_62.2</strong> (None)</dd>

<dd><strong>NODE_689_length_3764_cov_60.0</strong> (None)</dd>

<dd><strong>NODE_690_length_3762_cov_99.9</strong> (None)</dd>

<dd><strong>NODE_691_length_3752_cov_72.2</strong> (None)</dd>

<dd><strong>NODE_692_length_3751_cov_80.5</strong> (None)</dd>

<dd><strong>NODE_693_length_3749_cov_68.5</strong> (None)</dd>

<dd><strong>NODE_694_length_3749_cov_59.7</strong> (None)</dd>

<dd><strong>NODE_695_length_3748_cov_50.4</strong> (None)</dd>

<dd><strong>NODE_696_length_3748_cov_91.7</strong> (None)</dd>

<dd><strong>NODE_697_length_3745_cov_70.6</strong> (None)</dd>

<dd><strong>NODE_698_length_3742_cov_75.3</strong> (None)</dd>

<dd><strong>NODE_699_length_3742_cov_59.0</strong> (None)</dd>

<dd><strong>NODE_700_length_3734_cov_192.1</strong> (None)</dd>

<dd><strong>NODE_701_length_3733_cov_80.4</strong> (None)</dd>

<dd><strong>NODE_702_length_3730_cov_115.2</strong> (None)</dd>

<dd><strong>NODE_703_length_3728_cov_81.7</strong> (None)</dd>

<dd><strong>NODE_704_length_3723_cov_62.4</strong> (None)</dd>

<dd><strong>NODE_705_length_3722_cov_92.4</strong> (None)</dd>

<dd><strong>NODE_706_length_3722_cov_98.7</strong> (None)</dd>

<dd><strong>NODE_707_length_3721_cov_61.5</strong> (None)</dd>

<dd><strong>NODE_708_length_3717_cov_86.7</strong> (None)</dd>

<dd><strong>NODE_709_length_3710_cov_77.6</strong> (None)</dd>

<dd><strong>NODE_710_length_3707_cov_91.2</strong> (None)</dd>

<dd><strong>NODE_711_length_3707_cov_59.9</strong> (None)</dd>

<dd><strong>NODE_712_length_3706_cov_83.5</strong> (None)</dd>

<dd><strong>NODE_713_length_3700_cov_91.3</strong> (None)</dd>

<dd><strong>NODE_714_length_3700_cov_78.3</strong> (None)</dd>

<dd><strong>NODE_715_length_3698_cov_55.4</strong> (None)</dd>

<dd><strong>NODE_716_length_3697_cov_55.9</strong> (None)</dd>

<dd><strong>NODE_717_length_3692_cov_116.6</strong> (None)</dd>

<dd><strong>NODE_718_length_3691_cov_47.6</strong> (None)</dd>

<dd><strong>NODE_719_length_3691_cov_57.5</strong> (None)</dd>

<dd><strong>NODE_720_length_3689_cov_86.0</strong> (None)</dd>

<dd><strong>NODE_721_length_3686_cov_57.8</strong> (None)</dd>

<dd><strong>NODE_722_length_3682_cov_94.5</strong> (None)</dd>

<dd><strong>NODE_723_length_3680_cov_56.6</strong> (None)</dd>

<dd><strong>NODE_724_length_3679_cov_79.2</strong> (None)</dd>

<dd><strong>NODE_725_length_3678_cov_56.8</strong> (None)</dd>

<dd><strong>NODE_726_length_3676_cov_57.3</strong> (None)</dd>

<dd><strong>NODE_727_length_3672_cov_59.5</strong> (None)</dd>

<dd><strong>NODE_728_length_3670_cov_56.8</strong> (None)</dd>

<dd><strong>NODE_729_length_3669_cov_60.1</strong> (None)</dd>

<dd><strong>NODE_730_length_3669_cov_64.6</strong> (None)</dd>

<dd><strong>NODE_731_length_3668_cov_58.3</strong> (None)</dd>

<dd><strong>NODE_732_length_3667_cov_59.7</strong> (None)</dd>

<dd><strong>NODE_733_length_3667_cov_64.4</strong> (None)</dd>

<dd><strong>NODE_734_length_3664_cov_78.4</strong> (None)</dd>

<dd><strong>NODE_735_length_3663_cov_54.8</strong> (None)</dd>

<dd><strong>NODE_736_length_3663_cov_67.4</strong> (None)</dd>

<dd><strong>NODE_737_length_3662_cov_102.0</strong> (None)</dd>

<dd><strong>NODE_738_length_3661_cov_66.2</strong> (None)</dd>

<dd><strong>NODE_739_length_3656_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_740_length_3655_cov_75.3</strong> (None)</dd>

<dd><strong>NODE_741_length_3655_cov_51.6</strong> (None)</dd>

<dd><strong>NODE_742_length_3655_cov_66.5</strong> (None)</dd>

<dd><strong>NODE_743_length_3654_cov_99.7</strong> (None)</dd>

<dd><strong>NODE_744_length_3653_cov_56.1</strong> (None)</dd>

<dd><strong>NODE_745_length_3647_cov_56.6</strong> (None)</dd>

<dd><strong>NODE_746_length_3647_cov_57.9</strong> (None)</dd>

<dd><strong>NODE_747_length_3645_cov_58.4</strong> (None)</dd>

<dd><strong>NODE_748_length_3645_cov_53.0</strong> (None)</dd>

<dd><strong>NODE_749_length_3644_cov_88.6</strong> (None)</dd>

<dd><strong>NODE_750_length_3641_cov_84.7</strong> (None)</dd>

<dd><strong>NODE_751_length_3640_cov_59.1</strong> (None)</dd>

<dd><strong>NODE_752_length_3640_cov_56.1</strong> (None)</dd>

<dd><strong>NODE_753_length_3639_cov_46.2</strong> (None)</dd>

<dd><strong>NODE_754_length_3639_cov_108.1</strong> (None)</dd>

<dd><strong>NODE_755_length_3638_cov_52.1</strong> (None)</dd>

<dd><strong>NODE_756_length_3637_cov_57.1</strong> (None)</dd>

<dd><strong>NODE_757_length_3636_cov_86.1</strong> (None)</dd>

<dd><strong>NODE_758_length_3633_cov_52.2</strong> (None)</dd>

<dd><strong>NODE_759_length_3630_cov_54.3</strong> (None)</dd>

<dd><strong>NODE_760_length_3629_cov_98.2</strong> (None)</dd>

<dd><strong>NODE_761_length_3628_cov_87.5</strong> (None)</dd>

<dd><strong>NODE_762_length_3625_cov_68.4</strong> (None)</dd>

<dd><strong>NODE_763_length_3624_cov_77.7</strong> (None)</dd>

<dd><strong>NODE_764_length_3624_cov_58.8</strong> (None)</dd>

<dd><strong>NODE_765_length_3621_cov_79.1</strong> (None)</dd>

<dd><strong>NODE_766_length_3621_cov_69.6</strong> (None)</dd>

<dd><strong>NODE_767_length_3621_cov_63.1</strong> (None)</dd>

<dd><strong>NODE_768_length_3620_cov_105.4</strong> (None)</dd>

<dd><strong>NODE_769_length_3619_cov_76.3</strong> (None)</dd>

<dd><strong>NODE_770_length_3615_cov_61.1</strong> (None)</dd>

<dd><strong>NODE_771_length_3612_cov_53.1</strong> (None)</dd>

<dd><strong>NODE_772_length_3611_cov_70.9</strong> (None)</dd>

<dd><strong>NODE_773_length_3611_cov_84.0</strong> (None)</dd>

<dd><strong>NODE_774_length_3610_cov_79.2</strong> (None)</dd>

<dd><strong>NODE_775_length_3606_cov_82.7</strong> (None)</dd>

<dd><strong>NODE_776_length_3605_cov_77.7</strong> (None)</dd>

<dd><strong>NODE_777_length_3603_cov_67.9</strong> (None)</dd>

<dd><strong>NODE_778_length_3599_cov_101.3</strong> (None)</dd>

<dd><strong>NODE_779_length_3598_cov_57.5</strong> (None)</dd>

<dd><strong>NODE_780_length_3594_cov_53.3</strong> (None)</dd>

<dd><strong>NODE_781_length_3593_cov_76.7</strong> (None)</dd>

<dd><strong>NODE_782_length_3592_cov_57.9</strong> (None)</dd>

<dd><strong>NODE_783_length_3587_cov_85.2</strong> (None)</dd>

<dd><strong>NODE_784_length_3585_cov_81.6</strong> (None)</dd>

<dd><strong>NODE_785_length_3584_cov_57.4</strong> (None)</dd>

<dd><strong>NODE_786_length_3583_cov_60.1</strong> (None)</dd>

<dd><strong>NODE_787_length_3582_cov_53.8</strong> (None)</dd>

<dd><strong>NODE_788_length_3578_cov_97.4</strong> (None)</dd>

<dd><strong>NODE_789_length_3576_cov_102.0</strong> (None)</dd>

<dd><strong>NODE_790_length_3574_cov_99.8</strong> (None)</dd>

<dd><strong>NODE_791_length_3570_cov_63.1</strong> (None)</dd>

<dd><strong>NODE_792_length_3570_cov_54.3</strong> (None)</dd>

<dd><strong>NODE_793_length_3567_cov_55.7</strong> (None)</dd>

<dd><strong>NODE_794_length_3566_cov_83.4</strong> (None)</dd>

<dd><strong>NODE_795_length_3565_cov_63.9</strong> (None)</dd>

<dd><strong>NODE_796_length_3562_cov_65.8</strong> (None)</dd>

<dd><strong>NODE_797_length_3562_cov_98.3</strong> (None)</dd>

<dd><strong>NODE_798_length_3560_cov_82.2</strong> (None)</dd>

<dd><strong>NODE_799_length_3559_cov_82.9</strong> (None)</dd>

<dd><strong>NODE_800_length_3558_cov_94.0</strong> (None)</dd>

<dd><strong>NODE_801_length_3558_cov_70.6</strong> (None)</dd>

<dd><strong>NODE_802_length_3556_cov_65.1</strong> (None)</dd>

<dd><strong>NODE_803_length_3553_cov_72.4</strong> (None)</dd>

<dd><strong>NODE_804_length_3549_cov_112.4</strong> (None)</dd>

<dd><strong>NODE_805_length_3544_cov_56.0</strong> (None)</dd>

<dd><strong>NODE_806_length_3543_cov_55.6</strong> (None)</dd>

<dd><strong>NODE_807_length_3539_cov_73.2</strong> (None)</dd>

<dd><strong>NODE_808_length_3528_cov_67.0</strong> (None)</dd>

<dd><strong>NODE_809_length_3523_cov_106.0</strong> (None)</dd>

<dd><strong>NODE_810_length_3522_cov_95.5</strong> (None)</dd>

<dd><strong>NODE_811_length_3520_cov_79.2</strong> (None)</dd>

<dd><strong>NODE_812_length_3519_cov_69.2</strong> (None)</dd>

<dd><strong>NODE_813_length_3519_cov_85.8</strong> (None)</dd>

<dd><strong>NODE_814_length_3509_cov_57.8</strong> (None)</dd>

<dd><strong>NODE_815_length_3507_cov_124.2</strong> (None)</dd>

<dd><strong>NODE_816_length_3500_cov_64.5</strong> (None)</dd>

<dd><strong>NODE_817_length_3497_cov_62.7</strong> (None)</dd>

<dd><strong>NODE_818_length_3496_cov_60.5</strong> (None)</dd>

<dd><strong>NODE_819_length_3495_cov_61.7</strong> (None)</dd>

<dd><strong>NODE_820_length_3489_cov_63.6</strong> (None)</dd>

<dd><strong>NODE_821_length_3488_cov_56.0</strong> (None)</dd>

<dd><strong>NODE_822_length_3486_cov_53.0</strong> (None)</dd>

<dd><strong>NODE_823_length_3485_cov_58.1</strong> (None)</dd>

<dd><strong>NODE_824_length_3484_cov_82.4</strong> (None)</dd>

<dd><strong>NODE_825_length_3483_cov_53.8</strong> (None)</dd>

<dd><strong>NODE_826_length_3481_cov_51.4</strong> (None)</dd>

<dd><strong>NODE_827_length_3478_cov_65.4</strong> (None)</dd>

<dd><strong>NODE_828_length_3477_cov_87.1</strong> (None)</dd>

<dd><strong>NODE_829_length_3474_cov_106.8</strong> (None)</dd>

<dd><strong>NODE_830_length_3471_cov_91.2</strong> (None)</dd>

<dd><strong>NODE_831_length_3466_cov_54.6</strong> (None)</dd>

<dd><strong>NODE_832_length_3466_cov_57.8</strong> (None)</dd>

<dd><strong>NODE_833_length_3460_cov_57.8</strong> (None)</dd>

<dd><strong>NODE_834_length_3459_cov_65.4</strong> (None)</dd>

<dd><strong>NODE_835_length_3458_cov_61.1</strong> (None)</dd>

<dd><strong>NODE_836_length_3457_cov_83.0</strong> (None)</dd>

<dd><strong>NODE_837_length_3452_cov_59.3</strong> (None)</dd>

<dd><strong>NODE_838_length_3450_cov_76.9</strong> (None)</dd>

<dd><strong>NODE_839_length_3449_cov_62.1</strong> (None)</dd>

<dd><strong>NODE_840_length_3448_cov_53.7</strong> (None)</dd>

<dd><strong>NODE_841_length_3443_cov_97.5</strong> (None)</dd>

<dd><strong>NODE_842_length_3443_cov_92.8</strong> (None)</dd>

<dd><strong>NODE_843_length_3441_cov_58.9</strong> (None)</dd>

<dd><strong>NODE_844_length_3440_cov_59.5</strong> (None)</dd>

<dd><strong>NODE_845_length_3438_cov_58.8</strong> (None)</dd>

<dd><strong>NODE_846_length_3437_cov_65.9</strong> (None)</dd>

<dd><strong>NODE_847_length_3435_cov_62.7</strong> (None)</dd>

<dd><strong>NODE_848_length_3429_cov_81.5</strong> (None)</dd>

<dd><strong>NODE_849_length_3428_cov_102.6</strong> (None)</dd>

<dd><strong>NODE_850_length_3427_cov_53.2</strong> (None)</dd>

<dd><strong>NODE_851_length_3426_cov_56.4</strong> (None)</dd>

<dd><strong>NODE_852_length_3425_cov_57.2</strong> (None)</dd>

<dd><strong>NODE_853_length_3423_cov_86.2</strong> (None)</dd>

<dd><strong>NODE_854_length_3416_cov_61.9</strong> (None)</dd>

<dd><strong>NODE_855_length_3411_cov_63.1</strong> (None)</dd>

<dd><strong>NODE_856_length_3407_cov_84.7</strong> (None)</dd>

<dd><strong>NODE_857_length_3404_cov_80.2</strong> (None)</dd>

<dd><strong>NODE_858_length_3403_cov_62.9</strong> (None)</dd>

<dd><strong>NODE_859_length_3403_cov_87.8</strong> (None)</dd>

<dd><strong>NODE_860_length_3397_cov_99.0</strong> (None)</dd>

<dd><strong>NODE_861_length_3394_cov_56.7</strong> (None)</dd>

<dd><strong>NODE_862_length_3394_cov_98.8</strong> (None)</dd>

<dd><strong>NODE_863_length_3392_cov_90.4</strong> (None)</dd>

<dd><strong>NODE_864_length_3390_cov_57.7</strong> (None)</dd>

<dd><strong>NODE_865_length_3389_cov_90.2</strong> (None)</dd>

<dd><strong>NODE_866_length_3385_cov_83.5</strong> (None)</dd>

<dd><strong>NODE_867_length_3384_cov_104.8</strong> (None)</dd>

<dd><strong>NODE_868_length_3383_cov_87.6</strong> (None)</dd>

<dd><strong>NODE_869_length_3383_cov_95.5</strong> (None)</dd>

<dd><strong>NODE_870_length_3381_cov_52.8</strong> (None)</dd>

<dd><strong>NODE_871_length_3380_cov_71.5</strong> (None)</dd>

<dd><strong>NODE_872_length_3379_cov_75.1</strong> (None)</dd>

<dd><strong>NODE_873_length_3379_cov_59.3</strong> (None)</dd>

<dd><strong>NODE_874_length_3378_cov_54.5</strong> (None)</dd>

<dd><strong>NODE_875_length_3378_cov_71.4</strong> (None)</dd>

<dd><strong>NODE_876_length_3377_cov_53.6</strong> (None)</dd>

<dd><strong>NODE_877_length_3376_cov_79.0</strong> (None)</dd>

<dd><strong>NODE_878_length_3375_cov_56.5</strong> (None)</dd>

<dd><strong>NODE_879_length_3374_cov_77.9</strong> (None)</dd>

<dd><strong>NODE_880_length_3373_cov_52.5</strong> (None)</dd>

<dd><strong>NODE_881_length_3370_cov_104.6</strong> (None)</dd>

<dd><strong>NODE_882_length_3370_cov_93.7</strong> (None)</dd>

<dd><strong>NODE_883_length_3369_cov_97.4</strong> (None)</dd>

<dd><strong>NODE_884_length_3366_cov_96.9</strong> (None)</dd>

<dd><strong>NODE_885_length_3365_cov_69.5</strong> (None)</dd>

<dd><strong>NODE_886_length_3365_cov_52.7</strong> (None)</dd>

<dd><strong>NODE_887_length_3364_cov_63.0</strong> (None)</dd>

<dd><strong>NODE_888_length_3362_cov_57.6</strong> (None)</dd>

<dd><strong>NODE_889_length_3362_cov_52.8</strong> (None)</dd>

<dd><strong>NODE_890_length_3361_cov_55.5</strong> (None)</dd>

<dd><strong>NODE_891_length_3360_cov_85.4</strong> (None)</dd>

<dd><strong>NODE_892_length_3359_cov_55.1</strong> (None)</dd>

<dd><strong>NODE_893_length_3358_cov_72.9</strong> (None)</dd>

<dd><strong>NODE_894_length_3346_cov_60.0</strong> (None)</dd>

<dd><strong>NODE_895_length_3345_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_896_length_3344_cov_62.5</strong> (None)</dd>

<dd><strong>NODE_897_length_3342_cov_90.7</strong> (None)</dd>

<dd><strong>NODE_898_length_3342_cov_69.1</strong> (None)</dd>

<dd><strong>NODE_899_length_3342_cov_81.4</strong> (None)</dd>

<dd><strong>NODE_900_length_3341_cov_91.0</strong> (None)</dd>

<dd><strong>NODE_901_length_3340_cov_112.2</strong> (None)</dd>

<dd><strong>NODE_902_length_3340_cov_53.7</strong> (None)</dd>

<dd><strong>NODE_903_length_3338_cov_57.5</strong> (None)</dd>

<dd><strong>NODE_904_length_3336_cov_73.9</strong> (None)</dd>

<dd><strong>NODE_905_length_3335_cov_55.3</strong> (None)</dd>

<dd><strong>NODE_906_length_3335_cov_77.5</strong> (None)</dd>

<dd><strong>NODE_907_length_3334_cov_49.5</strong> (None)</dd>

<dd><strong>NODE_908_length_3332_cov_57.8</strong> (None)</dd>

<dd><strong>NODE_909_length_3329_cov_99.4</strong> (None)</dd>

<dd><strong>NODE_910_length_3329_cov_66.6</strong> (None)</dd>

<dd><strong>NODE_911_length_3329_cov_53.5</strong> (None)</dd>

<dd><strong>NODE_912_length_3325_cov_148.6</strong> (None)</dd>

<dd><strong>NODE_913_length_3322_cov_49.8</strong> (None)</dd>

<dd><strong>NODE_914_length_3320_cov_57.3</strong> (None)</dd>

<dd><strong>NODE_915_length_3318_cov_61.6</strong> (None)</dd>

<dd><strong>NODE_916_length_3317_cov_99.3</strong> (None)</dd>

<dd><strong>NODE_917_length_3313_cov_74.2</strong> (None)</dd>

<dd><strong>NODE_918_length_3307_cov_50.0</strong> (None)</dd>

<dd><strong>NODE_919_length_3305_cov_75.1</strong> (None)</dd>

<dd><strong>NODE_920_length_3303_cov_62.0</strong> (None)</dd>

<dd><strong>NODE_921_length_3302_cov_49.2</strong> (None)</dd>

<dd><strong>NODE_922_length_3300_cov_106.9</strong> (None)</dd>

<dd><strong>NODE_923_length_3300_cov_111.0</strong> (None)</dd>

<dd><strong>NODE_924_length_3300_cov_91.1</strong> (None)</dd>

<dd><strong>NODE_925_length_3299_cov_66.2</strong> (None)</dd>

<dd><strong>NODE_926_length_3299_cov_59.0</strong> (None)</dd>

<dd><strong>NODE_927_length_3299_cov_52.8</strong> (None)</dd>

<dd><strong>NODE_928_length_3298_cov_103.2</strong> (None)</dd>

<dd><strong>NODE_929_length_3297_cov_56.6</strong> (None)</dd>

<dd><strong>NODE_930_length_3290_cov_67.3</strong> (None)</dd>

<dd><strong>NODE_931_length_3290_cov_55.6</strong> (None)</dd>

<dd><strong>NODE_932_length_3289_cov_93.2</strong> (None)</dd>

<dd><strong>NODE_933_length_3287_cov_75.3</strong> (None)</dd>

<dd><strong>NODE_934_length_3287_cov_53.2</strong> (None)</dd>

<dd><strong>NODE_935_length_3286_cov_51.9</strong> (None)</dd>

<dd><strong>NODE_936_length_3286_cov_45.1</strong> (None)</dd>

<dd><strong>NODE_937_length_3285_cov_62.2</strong> (None)</dd>

<dd><strong>NODE_938_length_3283_cov_79.7</strong> (None)</dd>

<dd><strong>NODE_939_length_3281_cov_71.9</strong> (None)</dd>

<dd><strong>NODE_940_length_3281_cov_100.4</strong> (None)</dd>

<dd><strong>NODE_941_length_3280_cov_68.0</strong> (None)</dd>

<dd><strong>NODE_942_length_3280_cov_50.7</strong> (None)</dd>

<dd><strong>NODE_943_length_3279_cov_59.8</strong> (None)</dd>

<dd><strong>NODE_944_length_3278_cov_61.8</strong> (None)</dd>

<dd><strong>NODE_945_length_3278_cov_54.8</strong> (None)</dd>

<dd><strong>NODE_946_length_3277_cov_56.6</strong> (None)</dd>

<dd><strong>NODE_947_length_3276_cov_51.8</strong> (None)</dd>

<dd><strong>NODE_948_length_3273_cov_80.3</strong> (None)</dd>

<dd><strong>NODE_949_length_3272_cov_78.1</strong> (None)</dd>

<dd><strong>NODE_950_length_3267_cov_60.2</strong> (None)</dd>

<dd><strong>NODE_951_length_3267_cov_94.4</strong> (None)</dd>

<dd><strong>NODE_952_length_3266_cov_53.7</strong> (None)</dd>

<dd><strong>NODE_953_length_3265_cov_58.7</strong> (None)</dd>

<dd><strong>NODE_954_length_3265_cov_58.3</strong> (None)</dd>

<dd><strong>NODE_955_length_3264_cov_58.0</strong> (None)</dd>

<dd><strong>NODE_956_length_3263_cov_76.7</strong> (None)</dd>

<dd><strong>NODE_957_length_3263_cov_55.3</strong> (None)</dd>

<dd><strong>NODE_958_length_3263_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_959_length_3261_cov_87.7</strong> (None)</dd>

<dd><strong>NODE_960_length_3260_cov_51.2</strong> (None)</dd>

<dd><strong>NODE_961_length_3260_cov_55.9</strong> (None)</dd>

<dd><strong>NODE_962_length_3259_cov_56.1</strong> (None)</dd>

<dd><strong>NODE_963_length_3258_cov_54.6</strong> (None)</dd>

<dd><strong>NODE_964_length_3257_cov_47.5</strong> (None)</dd>

<dd><strong>NODE_965_length_3254_cov_81.0</strong> (None)</dd>

<dd><strong>NODE_966_length_3254_cov_55.4</strong> (None)</dd>

<dd><strong>NODE_967_length_3252_cov_60.7</strong> (None)</dd>

<dd><strong>NODE_968_length_3251_cov_61.3</strong> (None)</dd>

<dd><strong>NODE_969_length_3246_cov_56.6</strong> (None)</dd>

<dd><strong>NODE_970_length_3246_cov_91.8</strong> (None)</dd>

<dd><strong>NODE_971_length_3244_cov_54.1</strong> (None)</dd>

<dd><strong>NODE_972_length_3243_cov_53.6</strong> (None)</dd>

<dd><strong>NODE_973_length_3234_cov_99.2</strong> (None)</dd>

<dd><strong>NODE_974_length_3232_cov_54.0</strong> (None)</dd>

<dd><strong>NODE_975_length_3229_cov_65.7</strong> (None)</dd>

<dd><strong>NODE_976_length_3229_cov_77.8</strong> (None)</dd>

<dd><strong>NODE_977_length_3228_cov_48.3</strong> (None)</dd>

<dd><strong>NODE_978_length_3228_cov_54.5</strong> (None)</dd>

<dd><strong>NODE_979_length_3224_cov_81.6</strong> (None)</dd>

<dd><strong>NODE_980_length_3222_cov_54.5</strong> (None)</dd>

<dd><strong>NODE_981_length_3221_cov_122.1</strong> (None)</dd>

<dd><strong>NODE_982_length_3220_cov_69.8</strong> (None)</dd>

<dd><strong>NODE_983_length_3220_cov_87.0</strong> (None)</dd>

<dd><strong>NODE_984_length_3216_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_985_length_3215_cov_86.0</strong> (None)</dd>

<dd><strong>NODE_986_length_3213_cov_51.0</strong> (None)</dd>

<dd><strong>NODE_987_length_3210_cov_53.0</strong> (None)</dd>

<dd><strong>NODE_988_length_3209_cov_53.8</strong> (None)</dd>

<dd><strong>NODE_989_length_3208_cov_73.9</strong> (None)</dd>

<dd><strong>NODE_990_length_3207_cov_112.2</strong> (None)</dd>

<dd><strong>NODE_991_length_3205_cov_72.5</strong> (None)</dd>

<dd><strong>NODE_992_length_3205_cov_81.7</strong> (None)</dd>

<dd><strong>NODE_993_length_3203_cov_61.5</strong> (None)</dd>

<dd><strong>NODE_994_length_3202_cov_57.9</strong> (None)</dd>

<dd><strong>NODE_995_length_3201_cov_47.3</strong> (None)</dd>

<dd><strong>NODE_996_length_3195_cov_56.4</strong> (None)</dd>

<dd><strong>NODE_997_length_3194_cov_57.9</strong> (None)</dd>

<dd><strong>NODE_998_length_3191_cov_64.8</strong> (None)</dd>

<dd><strong>NODE_999_length_3189_cov_75.0</strong> (None)</dd>

<dd><strong>NODE_1000_length_3187_cov_63.3</strong> (None)</dd>

<dd><strong>NODE_1001_length_3185_cov_59.2</strong> (None)</dd>

<dd><strong>NODE_1002_length_3183_cov_78.1</strong> (None)</dd>

<dd><strong>NODE_1003_length_3180_cov_56.4</strong> (None)</dd>

<dd><strong>NODE_1004_length_3178_cov_53.7</strong> (None)</dd>

<dd><strong>NODE_1005_length_3176_cov_48.9</strong> (None)</dd>

<dd><strong>NODE_1006_length_3175_cov_62.8</strong> (None)</dd>

<dd><strong>NODE_1007_length_3170_cov_82.0</strong> (None)</dd>

<dd><strong>NODE_1008_length_3167_cov_50.5</strong> (None)</dd>

<dd><strong>NODE_1009_length_3164_cov_65.7</strong> (None)</dd>

<dd><strong>NODE_1010_length_3163_cov_86.0</strong> (None)</dd>

<dd><strong>NODE_1011_length_3162_cov_80.0</strong> (None)</dd>

<dd><strong>NODE_1012_length_3161_cov_61.1</strong> (None)</dd>

<dd><strong>NODE_1013_length_3159_cov_51.7</strong> (None)</dd>

<dd><strong>NODE_1014_length_3158_cov_76.7</strong> (None)</dd>

<dd><strong>NODE_1015_length_3158_cov_55.3</strong> (None)</dd>

<dd><strong>NODE_1016_length_3154_cov_85.5</strong> (None)</dd>

<dd><strong>NODE_1017_length_3152_cov_57.9</strong> (None)</dd>

<dd><strong>NODE_1018_length_3151_cov_57.4</strong> (None)</dd>

<dd><strong>NODE_1019_length_3150_cov_53.7</strong> (None)</dd>

<dd><strong>NODE_1020_length_3150_cov_62.0</strong> (None)</dd>

<dd><strong>NODE_1021_length_3148_cov_58.1</strong> (None)</dd>

<dd><strong>NODE_1022_length_3146_cov_58.1</strong> (None)</dd>

<dd><strong>NODE_1023_length_3146_cov_52.4</strong> (None)</dd>

<dd><strong>NODE_1024_length_3145_cov_60.4</strong> (None)</dd>

<dd><strong>NODE_1025_length_3145_cov_82.4</strong> (None)</dd>

<dd><strong>NODE_1026_length_3144_cov_64.5</strong> (None)</dd>

<dd><strong>NODE_1027_length_3143_cov_61.5</strong> (None)</dd>

<dd><strong>NODE_1028_length_3143_cov_70.0</strong> (None)</dd>

<dd><strong>NODE_1029_length_3142_cov_99.5</strong> (None)</dd>

<dd><strong>NODE_1030_length_3140_cov_54.4</strong> (None)</dd>

<dd><strong>NODE_1031_length_3136_cov_57.2</strong> (None)</dd>

<dd><strong>NODE_1032_length_3134_cov_54.9</strong> (None)</dd>

<dd><strong>NODE_1033_length_3134_cov_57.1</strong> (None)</dd>

<dd><strong>NODE_1034_length_3132_cov_72.2</strong> (None)</dd>

<dd><strong>NODE_1035_length_3132_cov_56.1</strong> (None)</dd>

<dd><strong>NODE_1036_length_3128_cov_77.5</strong> (None)</dd>

<dd><strong>NODE_1037_length_3126_cov_67.8</strong> (None)</dd>

<dd><strong>NODE_1038_length_3125_cov_91.0</strong> (None)</dd>

<dd><strong>NODE_1039_length_3124_cov_51.7</strong> (None)</dd>

<dd><strong>NODE_1040_length_3123_cov_60.1</strong> (None)</dd>

<dd><strong>NODE_1041_length_3123_cov_73.6</strong> (None)</dd>

<dd><strong>NODE_1042_length_3121_cov_57.6</strong> (None)</dd>

<dd><strong>NODE_1043_length_3119_cov_55.9</strong> (None)</dd>

<dd><strong>NODE_1044_length_3118_cov_80.7</strong> (None)</dd>

<dd><strong>NODE_1045_length_3117_cov_88.9</strong> (None)</dd>

<dd><strong>NODE_1046_length_3113_cov_55.4</strong> (None)</dd>

<dd><strong>NODE_1047_length_3113_cov_62.7</strong> (None)</dd>

<dd><strong>NODE_1048_length_3112_cov_51.1</strong> (None)</dd>

<dd><strong>NODE_1049_length_3112_cov_51.4</strong> (None)</dd>

<dd><strong>NODE_1050_length_3110_cov_73.6</strong> (None)</dd>

<dd><strong>NODE_1051_length_3105_cov_57.8</strong> (None)</dd>

<dd><strong>NODE_1052_length_3104_cov_58.7</strong> (None)</dd>

<dd><strong>NODE_1053_length_3100_cov_65.9</strong> (None)</dd>

<dd><strong>NODE_1054_length_3099_cov_55.5</strong> (None)</dd>

<dd><strong>NODE_1055_length_3098_cov_53.8</strong> (None)</dd>

<dd><strong>NODE_1056_length_3097_cov_65.1</strong> (None)</dd>

<dd><strong>NODE_1057_length_3097_cov_57.6</strong> (None)</dd>

<dd><strong>NODE_1058_length_3095_cov_94.1</strong> (None)</dd>

<dd><strong>NODE_1059_length_3093_cov_82.3</strong> (None)</dd>

<dd><strong>NODE_1060_length_3089_cov_80.6</strong> (None)</dd>

<dd><strong>NODE_1061_length_3089_cov_95.7</strong> (None)</dd>

<dd><strong>NODE_1062_length_3088_cov_62.3</strong> (None)</dd>

<dd><strong>NODE_1063_length_3087_cov_67.1</strong> (None)</dd>

<dd><strong>NODE_1064_length_3085_cov_49.5</strong> (None)</dd>

<dd><strong>NODE_1065_length_3084_cov_83.5</strong> (None)</dd>

<dd><strong>NODE_1066_length_3084_cov_62.3</strong> (None)</dd>

<dd><strong>NODE_1067_length_3082_cov_74.7</strong> (None)</dd>

<dd><strong>NODE_1068_length_3081_cov_53.7</strong> (None)</dd>

<dd><strong>NODE_1069_length_3079_cov_57.6</strong> (None)</dd>

<dd><strong>NODE_1070_length_3078_cov_55.2</strong> (None)</dd>

<dd><strong>NODE_1071_length_3077_cov_108.6</strong> (None)</dd>

<dd><strong>NODE_1072_length_3076_cov_55.3</strong> (None)</dd>

<dd><strong>NODE_1073_length_3075_cov_66.0</strong> (None)</dd>

<dd><strong>NODE_1074_length_3074_cov_71.4</strong> (None)</dd>

<dd><strong>NODE_1075_length_3070_cov_51.6</strong> (None)</dd>

<dd><strong>NODE_1076_length_3069_cov_97.1</strong> (None)</dd>

<dd><strong>NODE_1077_length_3069_cov_57.6</strong> (None)</dd>

<dd><strong>NODE_1078_length_3069_cov_70.8</strong> (None)</dd>

<dd><strong>NODE_1079_length_3067_cov_78.2</strong> (None)</dd>

<dd><strong>NODE_1080_length_3067_cov_63.9</strong> (None)</dd>

<dd><strong>NODE_1081_length_3067_cov_64.7</strong> (None)</dd>

<dd><strong>NODE_1082_length_3064_cov_59.7</strong> (None)</dd>

<dd><strong>NODE_1083_length_3062_cov_78.8</strong> (None)</dd>

<dd><strong>NODE_1084_length_3062_cov_96.3</strong> (None)</dd>

<dd><strong>NODE_1085_length_3062_cov_88.4</strong> (None)</dd>

<dd><strong>NODE_1086_length_3061_cov_63.2</strong> (None)</dd>

<dd><strong>NODE_1087_length_3059_cov_51.8</strong> (None)</dd>

<dd><strong>NODE_1088_length_3058_cov_53.0</strong> (None)</dd>

<dd><strong>NODE_1089_length_3055_cov_54.8</strong> (None)</dd>

<dd><strong>NODE_1090_length_3054_cov_67.3</strong> (None)</dd>

<dd><strong>NODE_1091_length_3054_cov_59.6</strong> (None)</dd>

<dd><strong>NODE_1092_length_3053_cov_67.7</strong> (None)</dd>

<dd><strong>NODE_1093_length_3053_cov_54.5</strong> (None)</dd>

<dd><strong>NODE_1094_length_3052_cov_70.5</strong> (None)</dd>

<dd><strong>NODE_1095_length_3052_cov_53.8</strong> (None)</dd>

<dd><strong>NODE_1096_length_3052_cov_55.5</strong> (None)</dd>

<dd><strong>NODE_1097_length_3049_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_1098_length_3048_cov_86.1</strong> (None)</dd>

<dd><strong>NODE_1099_length_3045_cov_70.7</strong> (None)</dd>

<dd><strong>NODE_1100_length_3045_cov_55.8</strong> (None)</dd>

<dd><strong>NODE_1101_length_3042_cov_63.2</strong> (None)</dd>

<dd><strong>NODE_1102_length_3042_cov_94.0</strong> (None)</dd>

<dd><strong>NODE_1103_length_3037_cov_87.1</strong> (None)</dd>

<dd><strong>NODE_1104_length_3037_cov_94.7</strong> (None)</dd>

<dd><strong>NODE_1105_length_3037_cov_55.0</strong> (None)</dd>

<dd><strong>NODE_1106_length_3036_cov_59.9</strong> (None)</dd>

<dd><strong>NODE_1107_length_3036_cov_61.7</strong> (None)</dd>

<dd><strong>NODE_1108_length_3033_cov_53.4</strong> (None)</dd>

<dd><strong>NODE_1109_length_3029_cov_75.5</strong> (None)</dd>

<dd><strong>NODE_1110_length_3028_cov_91.0</strong> (None)</dd>

<dd><strong>NODE_1111_length_3027_cov_97.7</strong> (None)</dd>

<dd><strong>NODE_1112_length_3024_cov_91.7</strong> (None)</dd>

<dd><strong>NODE_1113_length_3023_cov_65.2</strong> (None)</dd>

<dd><strong>NODE_1114_length_3023_cov_65.4</strong> (None)</dd>

<dd><strong>NODE_1115_length_3020_cov_57.7</strong> (None)</dd>

<dd><strong>NODE_1116_length_3017_cov_100.9</strong> (None)</dd>

<dd><strong>NODE_1117_length_3017_cov_77.4</strong> (None)</dd>

<dd><strong>NODE_1118_length_3017_cov_129.1</strong> (None)</dd>

<dd><strong>NODE_1119_length_3015_cov_76.9</strong> (None)</dd>

<dd><strong>NODE_1120_length_3015_cov_116.6</strong> (None)</dd>

<dd><strong>NODE_1121_length_3015_cov_67.0</strong> (None)</dd>

<dd><strong>NODE_1122_length_3015_cov_51.3</strong> (None)</dd>

<dd><strong>NODE_1123_length_3010_cov_74.4</strong> (None)</dd>

<dd><strong>NODE_1124_length_3010_cov_48.0</strong> (None)</dd>

<dd><strong>NODE_1125_length_3009_cov_75.2</strong> (None)</dd>

<dd><strong>NODE_1126_length_3008_cov_58.3</strong> (None)</dd>

<dd><strong>NODE_1127_length_3007_cov_49.4</strong> (None)</dd>

<dd><strong>NODE_1128_length_3006_cov_81.2</strong> (None)</dd>

<dd><strong>NODE_1129_length_3005_cov_63.3</strong> (None)</dd>

<dd><strong>NODE_1130_length_3005_cov_66.2</strong> (None)</dd>

<dd><strong>NODE_1131_length_3002_cov_53.6</strong> (None)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_3_length_18813_cov_105.5 - Region 1 - RRE-containing <div class="help-container"> <div class="help-icon" data-id="region_svg-help-57"></div> <span class="help-tooltip" id="region_svg-help-57">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_3_length_18813_cov_105.5.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="NODE_3_length_18813_cov_105.5_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,928 - 18,813 nt. (total: 11,886 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r3c1-rules">RRE-containing: minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE])</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-58"></div> <span class="help-tooltip" id="focus-panel-help-58">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c1-tfbs-finder" data-name="r3c1-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-8"></div> <span class="help-tooltip" id="gt-help-tooltip-help-8">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RRE-containing</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>ectoine</td>


<td>Methylomicrobium kenyense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002306: 0-3536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002306/index.html#r1c1" target="_new">BGC0002306</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>freyrasin</td>


<td>Paenibacillus polymyxa ATCC 842</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001239: 107-6956">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001239/index.html#r1c1" target="_new">BGC0001239</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>biotin</td>


<td>Aspergillus nidulans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000707: 0-7321">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000707/index.html#r1c1" target="_new">BGC0000707</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Saccharide</td>

<td>kasugamycin</td>


<td>Streptomyces kasugaensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>NRP</td>

<td>fragin</td>


<td>Burkholderia cenocepacia H111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001488: 0-13752">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001488/index.html#r1c1" target="_new">BGC0001488</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Other</td>

<td>4-formylaminooxyvinylglycine</td>


<td>Pseudomonas fluorescens WH6</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905: 1-16359">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliobacillus mobilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliophilum fasciatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.41, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Other</td>

<td>schizokinen</td>


<td>Leptolyngbya sp. PCC 7376</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002186: 329-29262">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002186/index.html#r1c1" target="_new">BGC0002186</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>fusaristatin A</td>


<td>Fusarium graminearum PH-1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000855: 245-3455">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000855/index.html#r1c1" target="_new">BGC0000855</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>ectoine</td>
<td>Methylomicrobium kenyense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002306: 0-3536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002306/index.html#r1c1" target="_new">BGC0002306</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>RiPP</td>
<td>freyrasin</td>
<td>Paenibacillus polymyxa ATCC 842</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001239: 107-6956">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001239/index.html#r1c1" target="_new">BGC0001239</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>biotin</td>
<td>Aspergillus nidulans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000707: 0-7321">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000707/index.html#r1c1" target="_new">BGC0000707</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Saccharide</td>
<td>kasugamycin</td>
<td>Streptomyces kasugaensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>NRP</td>
<td>fragin</td>
<td>Burkholderia cenocepacia H111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001488: 0-13752">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001488/index.html#r1c1" target="_new">BGC0001488</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Other</td>
<td>4-formylaminooxyvinylglycine</td>
<td>Pseudomonas fluorescens WH6</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905: 1-16359">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Other</td>
<td>heme D1</td>
<td>Heliobacillus mobilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Other</td>
<td>heme D1</td>
<td>Heliophilum fasciatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.41, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Other</td>
<td>schizokinen</td>
<td>Leptolyngbya sp. PCC 7376</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002186: 329-29262">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002186/index.html#r1c1" target="_new">BGC0002186</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP, Polyketide</td>
<td>fusaristatin A</td>
<td>Fusarium graminearum PH-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-2"></div> <span class="help-tooltip" id="cb-general-help-2">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c1_1.svg">
NZ_CGIH01000004 (36444-58073): Syntrophomonas zehnderi OL-4, whole genome sho... (16% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_2.svg">
NZ_DHSN01000054 (16008-37600): MAG: Syntrophomonas wolfei isolate UBA5285 UBA... (12% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_3.svg">
NC_008346 (2794787-2816379): Syntrophomonas wolfei subsp. wolfei str. Goettin... (15% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_4.svg">
NZ_CP046457 (2103559-2125138): Candidatus Syntrophocurvum alkaliphilum isolat... (12% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_5.svg">
NC_014220 (2346892-2368669): Syntrophothermus lipocalidus DSM 12680, complete... (20% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_6.svg">
NC_015565 (2848707-2870396): Desulfotomaculum nigrificans CO-1-SRB, complete ... (15% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_7.svg">
NZ_KI912183 (2251047-2272736): Desulfotomaculum nigrificans DSM 574 DesniDRAF... (15% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_8.svg">
NZ_CP019698 (75419-97039): Desulforamulus ferrireducens strain GSS09 chromoso... (20% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_9.svg">
NC_009253 (3568679-3590325): Desulforamulus reducens MI-1, complete sequence (15% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r3c1_10.svg">
NZ_FRAR01000016 (51785-73425): Desulforamulus aeronauticus DSM 10349, whole g... (15% of genes show similarity), ranthipeptide
</option>

</select>
<button id= "clusterblast-r3c1-download">Download graphic</button>
</div>
<div id="clusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-3"></div> <span class="help-tooltip" id="cb-known-help-3">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-4"></div> <span class="help-tooltip" id="cb-sub-help-4">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-7"></div> <span class="help-tooltip" id="tfbs-body-help-7">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 21.31 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-8003-ZuR"></div>

</div>


</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r3c1-RREfinder" data-name="r3c1-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r3c1-tfbs-finder" data-name="r3c1-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r3c1-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-5"></div> <span class="help-tooltip" id="rre-side-help-5">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>RRE-containing protocluster</strong> (6927...18813)</span>
<dt></dt>




<strong>NCFMKG_00335</strong> <div class="collapser collapser-target-NCFMKG_00335 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: Ranthipeptide_rSAM_RRE</dd>
<dd>Location: 26 - 124</dd>
<dd>Score: 60.0</dd>

<dd>RRE type: Quinohemoprotein_rSAM_RRE</dd>
<dd>Location: 25 - 97</dd>
<dd>Score: 26.1</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r3c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-6"></div> <span class="help-tooltip" id="tfbs-side-help-6">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>





<div class="page" id='r15c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_15_length_13587_cov_90.8 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-59"></div> <span class="help-tooltip" id="region_svg-help-59">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_15_length_13587_cov_90.8.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r15c1-svg" data-filename="NODE_15_length_13587_cov_90.8_r15c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 13,587 nt. (total: 13,587 nt)

<a class="cluster-rules-header" id="r15c1-rules-header" href="#r15c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r15c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r15c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-60"></div> <span class="help-tooltip" id="focus-panel-help-60">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r15c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r15c1-gene-table-details" data-name="r15c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r15c1-nrps_pks" data-name="r15c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r15c1-MIBiG-cluster-compare" data-name="r15c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r15c1-clusterblast" data-name="r15c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r15c1-knownclusterblast" data-name="r15c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r15c1-subclusterblast" data-name="r15c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r15c1-tfbs-finder" data-name="r15c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r15c1-nrps-pks-bubbles" data-name="r15c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r15c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-19"></div> <span class="help-tooltip" id="gt-help-tooltip-help-19">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r15c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r15c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r15c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r15c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r15c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-9"></div> <span class="help-tooltip" id="nrps-domain-help-9">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r15c1-details-svg" data-filename="NODE_15_length_13587_cov_90.8_r15c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r15c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r15c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-10"></div> <span class="help-tooltip" id="MIBiG-help-10">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r15c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r15c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r15c1"></div>
<div id="comparison-MIBiG-r15c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>AbT1</td>


<td>Aureobasidium pullulans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>ambactin</td>


<td>Xenorhabdus miraniensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r15c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>AbT1</td>
<td>Aureobasidium pullulans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001131: 0-29471">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001131/index.html#r1c1" target="_new">BGC0001131</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>NRP</td>
<td>ambactin</td>
<td>Xenorhabdus miraniensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r15c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-11"></div> <span class="help-tooltip" id="cb-general-help-11">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r15c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r15c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r15c1_1.svg">
NZ_VLKC01000002 (1314951-1379477): Sporomusa sp. KB1 SalpaDRAFT Scaffold1.2, ... (19% of genes show similarity), NRP-metallophore,NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r15c1_2.svg">
NZ_CTRP01000014 (327144-383971): Sporomusa ovata isolate Sporomusa ovata stra... (11% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r15c1_3.svg">
NZ_CP010978 (4000091-4062389): Pelosinus fermentans JBW45 chromosome, complet... (13% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r15c1_4.svg">
NC_015738 (20076-75386): Eggerthella sp. YY7918, complete sequence (12% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r15c1_5.svg">
NZ_CP033719 (645337-700932): Propionibacterium acidifaciens strain FDAARGOS 5... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r15c1_6.svg">
NZ_BMMP01000003 (210440-260976): Streptomyces daqingensis strain CGMCC 4.7178... (13% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r15c1_7.svg">
NZ_CZBP01000007 (75289-137286): Blautia obeum strain 2789STDY5834957, whole g... (15% of genes show similarity), NRPS,cyclic-lactone-autoinducer
</option>

<option value = "svg/clusterblast_r15c1_8.svg">
NZ_QULA01000001 (313823-375820): Blautia sp. OM07-19 OM07-19.Scaf1, whole gen... (15% of genes show similarity), NRPS,cyclic-lactone-autoinducer
</option>

<option value = "svg/clusterblast_r15c1_9.svg">
NZ_KB933398 (924179-987827): Lysinibacillus sphaericus OT4b.31 Ls31 scaffold1... (11% of genes show similarity), NRPS,betalactone
</option>

<option value = "svg/clusterblast_r15c1_10.svg">
NZ_CP009438 (990287-1105128): Streptomyces glaucescens strain GLA.O chromosom... (8% of genes show similarity), NRP-metallophore,NRPS,NRPS-like
</option>

</select>
<button id= "clusterblast-r15c1-download">Download graphic</button>
</div>
<div id="clusterblast-r15c1-svg"></div>

</div>
</div>

<div class="body-details-section r15c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-12"></div> <span class="help-tooltip" id="cb-known-help-12">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r15c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-13"></div> <span class="help-tooltip" id="cb-sub-help-13">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r15c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-17"></div> <span class="help-tooltip" id="tfbs-body-help-17">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 19.39 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r15c1-12606-CatR"></div>

</div>


</div>
</div>



<div class="body-details-section r15c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-18"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-18">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r15c1-domain-bubble-svg" data-filename="NODE_15_length_13587_cov_90.8_r15c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r15c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 13587): single NRPS
</option>


</select>

<div class="details-svg" id="r15c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r15c1-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r15c1-nrps_pks_products" data-name="r15c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r15c1-nrps_pks_monomers" data-name="r15c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r15c1-tfbs-finder" data-name="r15c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r15c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-14"></div> <span class="help-tooltip" id="nrps-structures-help-14">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 13587:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r15c1-1" class="smiles-canvas" data-smiles="NC(C(C)S)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-Cys)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-cys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-cys*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r15c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C(C)S)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r15c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-15"></div> <span class="help-tooltip" id="nrps-monomers-help-15">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">NCFMKG_01150</span></strong>: Cys
<div class="collapser collapser-target-NCFMKG_01150 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (530..936)</strong>: Cys
<div class="collapser collapser-target-NCFMKG_01150 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Cys
<div class="collapser collapser-target-NCFMKG_01150-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
<dt>Small clusters prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
<dt>Single AA prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLYNLSLIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Cys <span class="serif">DLYNLSLIWK</span> (88% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r15c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-16"></div> <span class="help-tooltip" id="tfbs-side-help-16">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r72c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_72_length_8481_cov_79.8 - Region 1 - ranthipeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-61"></div> <span class="help-tooltip" id="region_svg-help-61">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_72_length_8481_cov_79.8.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r72c1-svg" data-filename="NODE_72_length_8481_cov_79.8_r72c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 8,481 nt. (total: 8,481 nt)

<a class="cluster-rules-header" id="r72c1-rules-header" href="#r72c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r72c1-rules">ranthipeptide: (TIGR03973 or SCIFF or (cds(PF04055 and (TIGR04085 or SPASM)) and minimum(1, [Bottromycin_Methyltransferase_RRE, Cyanobactin_D_RRE, Cytolysin_C_RRE, Goadsporin_CD_RRE, Heterocycloanthracin_C_RRE, Lanthipeptide_LanB_RRE, Lasso_Fused_RRE, Mycofactocin_RRE, NHLP_CD_RRE, Other_Sactipeptide_rSAM_RRE, Pantocin_Microcin_RRE, Plantazolicin_C_RRE, PlpY_RRE, PqqD_RRE, Proteusin_Epimerase_RRE, Proteusin_Methyltransferase_RRE, Quinohemoprotein_rSAM_RRE, Ranthipeptide_rSAM_RRE, Stand_Alone_Lasso_RRE, Streptide_RRE, Subtilosin_rSAM_RRE, Thiaglutamate_B_RRE, Thiaglutamate_G_RRE, Thiaglutamate_I_RRE, Thiopeptide_F_RRE, Thuricin_rSAM_RRE, Thurincin_rSAM_RRE, Trifolitoxin_RRE]) and not (TIGR01716 or subtilosin or TIGR02111 or TIGR03966 or TIGR04103 or (YcaO or TIGR03604))))</div>

<div class="region-svg-container">
<div id='r72c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-62"></div> <span class="help-tooltip" id="focus-panel-help-62">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r72c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r72c1-gene-table-details" data-name="r72c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r72c1-MIBiG-cluster-compare" data-name="r72c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r72c1-clusterblast" data-name="r72c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r72c1-knownclusterblast" data-name="r72c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r72c1-subclusterblast" data-name="r72c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r72c1-tfbs-finder" data-name="r72c1-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r72c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-26"></div> <span class="help-tooltip" id="gt-help-tooltip-help-26">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r72c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r72c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r72c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r72c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r72c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-20"></div> <span class="help-tooltip" id="MIBiG-help-20">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r72c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r72c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r72c1"></div>
<div id="comparison-MIBiG-r72c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">ranthipeptide</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002306: 0-3536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002306/index.html#r1c1" target="_new">BGC0002306</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>freyrasin</td>


<td>Paenibacillus polymyxa ATCC 842</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000695: 0-47035">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000695/index.html#r1c1" target="_new">BGC0000695</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Saccharide</td>

<td>fortimicin</td>


<td>Micromonospora olivasterospora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000651: 740-35007">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000651/index.html#r1c1" target="_new">BGC0000651</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.45, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.04</td>
<td>Terpene</td>

<td>KS-505a</td>


<td>Streptomyces argenteolus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000689: 101-32666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000689/index.html#r1c1" target="_new">BGC0000689</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.04</td>
<td>Saccharide</td>

<td>2-deoxystreptamine</td>


<td>Micromonospora echinospora</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r72c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002306: 0-3536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002306/index.html#r1c1" target="_new">BGC0002306</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>freyrasin</td>
<td>Paenibacillus polymyxa ATCC 842</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000695: 0-47035">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000695/index.html#r1c1" target="_new">BGC0000695</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Saccharide</td>
<td>fortimicin</td>
<td>Micromonospora olivasterospora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000651: 740-35007">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000651/index.html#r1c1" target="_new">BGC0000651</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.45, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>Terpene</td>
<td>KS-505a</td>
<td>Streptomyces argenteolus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000689: 101-32666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000689/index.html#r1c1" target="_new">BGC0000689</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.44, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>Saccharide</td>
<td>2-deoxystreptamine</td>
<td>Micromonospora echinospora</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r72c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-21"></div> <span class="help-tooltip" id="cb-general-help-21">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r72c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r72c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r72c1_1.svg">
NC_013895 (893618-915188): Mageeibacillus indolicus UPII9-5, complete sequence (35% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_2.svg">
NZ_NIOI01000001 (1328303-1350127): Pseudoclostridium thermosuccinogenes strai... (27% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_3.svg">
NZ_JHYD01000009 (176518-198200): Ruminiclostridium cellobioparum DSM 1351 = A... (31% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_4.svg">
NZ_CP119677 (1788662-1810286): Ruminiclostridium papyrosolvens DSM 2782 chrom... (28% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_5.svg">
NZ_CP061336 (2623908-2645630): Ruminiclostridium herbifermentans strain MA18 ... (33% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_6.svg">
NC_016791 (1817302-1838926): Clostridium sp. BNL1100, complete sequence (28% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_7.svg">
NC_011898 (609979-631602): Ruminiclostridium cellulolyticum H10, complete seq... (28% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_8.svg">
NZ_CP021850 (1701761-1723585): Pseudoclostridium thermosuccinogenes strain DS... (25% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_9.svg">
NZ_QPJT01000016 (41599-63266): Anaerobacterium chartisolvens strain DSM 27016... (25% of genes show similarity), ranthipeptide
</option>

<option value = "svg/clusterblast_r72c1_10.svg">
NZ_NIBG01000001 (39610-61298): Anaeromicrobium sediminis strain DY2726D Seq1,... (20% of genes show similarity), ranthipeptide
</option>

</select>
<button id= "clusterblast-r72c1-download">Download graphic</button>
</div>
<div id="clusterblast-r72c1-svg"></div>

</div>
</div>

<div class="body-details-section r72c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-22"></div> <span class="help-tooltip" id="cb-known-help-22">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r72c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-23"></div> <span class="help-tooltip" id="cb-sub-help-23">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r72c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-25"></div> <span class="help-tooltip" id="tfbs-body-help-25">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 19.79 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r72c1-6261-CatR"></div>

</div>


</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r72c1-tfbs-finder" data-name="r72c1-tfbs-finder">
TFBS Finder
</div>


</div>










<div class="sidepanel-details-section r72c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-24"></div> <span class="help-tooltip" id="tfbs-side-help-24">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>





<div class="page" id='r106c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_106_length_7571_cov_86.5 - Region 1 - phosphonate <div class="help-container"> <div class="help-icon" data-id="region_svg-help-63"></div> <span class="help-tooltip" id="region_svg-help-63">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_106_length_7571_cov_86.5.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r106c1-svg" data-filename="NODE_106_length_7571_cov_86.5_r106c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 7,571 nt. (total: 7,571 nt)

<a class="cluster-rules-header" id="r106c1-rules-header" href="#r106c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r106c1-rules">phosphonate: (phosphonates and minimum(1, [2-Hacid_dh_C, Aldedh, Aminotran_1_2, Aminotran_5, AurF, CTP_transf_like, DUF4992, Fe-ADH, HMGL-like, Metallophos_2, NTP_transf_3, NTP_transf_5, PALP, PEP_mutase, TPP_enzyme_C, TPP_enzyme_N, phosphonates]) and not (phosphonatesHydrolase or phosphonatesMeIsocitrateLyase))</div>

<div class="region-svg-container">
<div id='r106c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-64"></div> <span class="help-tooltip" id="focus-panel-help-64">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r106c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r106c1-gene-table-details" data-name="r106c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r106c1-MIBiG-cluster-compare" data-name="r106c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r106c1-clusterblast" data-name="r106c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r106c1-knownclusterblast" data-name="r106c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r106c1-subclusterblast" data-name="r106c1-subclusterblast">
SubClusterBlast
</div>




</div>


<div class="body-details-section r106c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-31"></div> <span class="help-tooltip" id="gt-help-tooltip-help-31">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r106c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r106c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r106c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r106c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r106c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-27"></div> <span class="help-tooltip" id="MIBiG-help-27">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r106c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r106c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r106c1"></div>
<div id="comparison-MIBiG-r106c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">phosphonate</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>FR-900098</td>


<td>Streptomyces rubellomurinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000897: 0-20143">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000897/index.html#r1c1" target="_new">BGC0000897</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Other</td>

<td>dehydrophos</td>


<td>Streptomyces luridus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001739: 0-14815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001739/index.html#r1c1" target="_new">BGC0001739</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.46, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Other</td>

<td>phosphonoacetic Acid</td>


<td>Streptomyces sp. NRRL F-525</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.44, order:0.30, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Other</td>

<td>fosfazinomycin A, fosfazinomycin B</td>


<td>Streptomyces sp. WM6372</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000926: 1-12724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000926/index.html#r1c1" target="_new">BGC0000926</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Other</td>

<td>rhizocticin A</td>


<td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002036: 0-15983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002036/index.html#r1c1" target="_new">BGC0002036</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.45, order:0.30, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Other</td>

<td>dehydrofosmidomycin</td>


<td>Streptomyces lavendulae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000938: 104-31263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000938/index.html#r1c1" target="_new">BGC0000938</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.48, order:0.30, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Other (Aminocoumarin)</td>

<td>fosfomycin</td>


<td>Streptomyces fradiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001411: 221-34214">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001411/index.html#r1c1" target="_new">BGC0001411</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.49, order:0.30, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Saccharide</td>

<td>polysaccharide B</td>


<td>Bacteroides fragilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000807: 0-36089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000807/index.html#r1c1" target="_new">BGC0000807</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.48, order:0.30, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Saccharide</td>

<td>phosphonoglycans</td>


<td>Stackebrandtia nassauensis DSM 44728</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000406: 116-37823">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000406/index.html#r1c1" target="_new">BGC0000406</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.44, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>NRP</td>

<td>phosphinothricintripeptide</td>


<td>Streptomyces viridochromogenes</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r106c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000904: 116-14341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000904/index.html#r1c1" target="_new">BGC0000904</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>FR-900098</td>
<td>Streptomyces rubellomurinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000897: 0-20143">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000897/index.html#r1c1" target="_new">BGC0000897</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Other</td>
<td>dehydrophos</td>
<td>Streptomyces luridus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001739: 0-14815">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001739/index.html#r1c1" target="_new">BGC0001739</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.46, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Other</td>
<td>phosphonoacetic Acid</td>
<td>Streptomyces sp. NRRL F-525</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000937: 0-23647">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000937/index.html#r1c1" target="_new">BGC0000937</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.44, order:0.30, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Other</td>
<td>fosfazinomycin A, fosfazinomycin B</td>
<td>Streptomyces sp. WM6372</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000926: 1-12724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000926/index.html#r1c1" target="_new">BGC0000926</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.46, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Other</td>
<td>rhizocticin A</td>
<td>Bacillus subtilis subsp. spizizenii ATCC 6633</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002036: 0-15983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002036/index.html#r1c1" target="_new">BGC0002036</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.45, order:0.30, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Other</td>
<td>dehydrofosmidomycin</td>
<td>Streptomyces lavendulae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000938: 104-31263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000938/index.html#r1c1" target="_new">BGC0000938</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.48, order:0.30, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Other (Aminocoumarin)</td>
<td>fosfomycin</td>
<td>Streptomyces fradiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001411: 221-34214">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001411/index.html#r1c1" target="_new">BGC0001411</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.49, order:0.30, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Saccharide</td>
<td>polysaccharide B</td>
<td>Bacteroides fragilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000807: 0-36089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000807/index.html#r1c1" target="_new">BGC0000807</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.48, order:0.30, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Saccharide</td>
<td>phosphonoglycans</td>
<td>Stackebrandtia nassauensis DSM 44728</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000406: 116-37823">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000406/index.html#r1c1" target="_new">BGC0000406</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.44, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>NRP</td>
<td>phosphinothricintripeptide</td>
<td>Streptomyces viridochromogenes</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r106c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-28"></div> <span class="help-tooltip" id="cb-general-help-28">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r106c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r106c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r106c1_1.svg">
NZ_RKRG01000002 (295298-316605): Methanobrevibacter gottschalkii DSM 11977 Ga... (36% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_2.svg">
NZ_CACZYW010000005 (81104-101729): MAG: Eubacterium ruminantium isolate RUG13... (40% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_3.svg">
NZ_HG796200 (130987-149702): Methanobrevibacter oralis JMR01, whole genome sh... (37% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_4.svg">
NZ_AUKD01000018 (70300-99535): Butyrivibrio fibrisolvens MD2001 G635DRAFT sca... (31% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_5.svg">
NZ_CP092473 (499238-527505): Roseburia sp. NSJ-69 chromosome, complete genome (26% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_6.svg">
NC_021012 (1997100-2025859): Roseburia intestinalis XB6B4, complete sequence (29% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_7.svg">
NZ_FNUF01000001 (424193-445160): Prevotella sp. lc2012, whole genome shotgun ... (33% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_8.svg">
NZ_CP051197 (939460-967453): Treponema vincentii strain OMZ 802 chromosome, c... (24% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_9.svg">
NZ_KE332514 (78230-98093): Treponema vincentii F0403 aczIO-supercont1.3, whol... (35% of genes show similarity), phosphonate
</option>

<option value = "svg/clusterblast_r106c1_10.svg">
NZ_JADNCX010000011 (35274-66623): Bacteroides uniformis strain D31t1 170403 D... (21% of genes show similarity), phosphonate
</option>

</select>
<button id= "clusterblast-r106c1-download">Download graphic</button>
</div>
<div id="clusterblast-r106c1-svg"></div>

</div>
</div>

<div class="body-details-section r106c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-29"></div> <span class="help-tooltip" id="cb-known-help-29">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r106c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r106c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r106c1_1.svg">
polysaccharide B
</option>

</select>
<button id="knownclusterblast-r106c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r106c1-svg"></div>

</div>
</div>

<div class="body-details-section r106c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-30"></div> <span class="help-tooltip" id="cb-sub-help-30">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>




</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r259c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_259_length_5660_cov_107.3 - Region 1 - thioamitides <div class="help-container"> <div class="help-icon" data-id="region_svg-help-65"></div> <span class="help-tooltip" id="region_svg-help-65">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_259_length_5660_cov_107.3.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r259c1-svg" data-filename="NODE_259_length_5660_cov_107.3_r259c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 5,660 nt. (total: 5,660 nt)

<a class="cluster-rules-header" id="r259c1-rules-header" href="#r259c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r259c1-rules">thioamitides: ((YcaO or TIGR03604) and TfuA)</div>

<div class="region-svg-container">
<div id='r259c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-66"></div> <span class="help-tooltip" id="focus-panel-help-66">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r259c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r259c1-gene-table-details" data-name="r259c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r259c1-MIBiG-cluster-compare" data-name="r259c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r259c1-clusterblast" data-name="r259c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r259c1-knownclusterblast" data-name="r259c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r259c1-subclusterblast" data-name="r259c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r259c1-tfbs-finder" data-name="r259c1-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r259c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-38"></div> <span class="help-tooltip" id="gt-help-tooltip-help-38">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r259c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r259c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r259c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r259c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r259c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-32"></div> <span class="help-tooltip" id="MIBiG-help-32">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r259c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r259c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r259c1"></div>
<div id="comparison-MIBiG-r259c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">thioamitides</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001849: 0-6950">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001849/index.html#r1c1" target="_new">BGC0001849</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>RiPP</td>

<td>thiovarsolin A, thiovarsolin B, thiovarsolin C, thiovarsolin D</td>


<td>Streptomyces varsoviensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002560: 0-11740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002560/index.html#r1c1" target="_new">BGC0002560</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.44, order:0.45, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>thiostreptamide S4</td>


<td>Streptomyces sp. NRRL S-4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>corynazolicin</td>


<td>Corynebacterium urealyticum DSM 7111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002485: 143-14387">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002485/index.html#r1c1" target="_new">BGC0002485</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>RiPP, Alkaloid</td>

<td>muscoride A, muscoride B</td>


<td>Desmonostoc sp. PCC 7906</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002551: 210-13006">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002551/index.html#r1c1" target="_new">BGC0002551</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Alkaloid, RiPP, Terpene</td>

<td>muscoride A, muscoride B</td>


<td>Nostoc sp. UHCC-0398</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000483: 1302-13724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000483/index.html#r1c1" target="_new">BGC0000483</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>aeruginosamide B, aeruginosamide C</td>


<td>Microcystis aeruginosa PCC 9432</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000481: 0-12515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000481/index.html#r1c1" target="_new">BGC0000481</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>trichamide</td>


<td>Trichodesmium erythraeum IMS101</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001802: 854-10845">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001802/index.html#r1c1" target="_new">BGC0001802</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:0.45, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>thioholgamide A, thioholgamide B</td>


<td>Streptomyces malaysiense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001696: 0-9985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001696/index.html#r1c1" target="_new">BGC0001696</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:0.45, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>neothioviridamide</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002486: 0-10474">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002486/index.html#r1c1" target="_new">BGC0002486</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>tolypamide</td>


<td>Tolypothrix sp. PCC 7601</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r259c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001849: 0-6950">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001849/index.html#r1c1" target="_new">BGC0001849</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>RiPP</td>
<td>thiovarsolin A, thiovarsolin B, thiovarsolin C, thiovarsolin D</td>
<td>Streptomyces varsoviensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002560: 0-11740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002560/index.html#r1c1" target="_new">BGC0002560</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.44, order:0.45, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>thiostreptamide S4</td>
<td>Streptomyces sp. NRRL S-4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>corynazolicin</td>
<td>Corynebacterium urealyticum DSM 7111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002485: 143-14387">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002485/index.html#r1c1" target="_new">BGC0002485</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>RiPP, Alkaloid</td>
<td>muscoride A, muscoride B</td>
<td>Desmonostoc sp. PCC 7906</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002551: 210-13006">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002551/index.html#r1c1" target="_new">BGC0002551</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Alkaloid, RiPP, Terpene</td>
<td>muscoride A, muscoride B</td>
<td>Nostoc sp. UHCC-0398</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000483: 1302-13724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000483/index.html#r1c1" target="_new">BGC0000483</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>aeruginosamide B, aeruginosamide C</td>
<td>Microcystis aeruginosa PCC 9432</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000481: 0-12515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000481/index.html#r1c1" target="_new">BGC0000481</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>trichamide</td>
<td>Trichodesmium erythraeum IMS101</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001802: 854-10845">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001802/index.html#r1c1" target="_new">BGC0001802</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:0.45, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>thioholgamide A, thioholgamide B</td>
<td>Streptomyces malaysiense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001696: 0-9985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001696/index.html#r1c1" target="_new">BGC0001696</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:0.45, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>neothioviridamide</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002486: 0-10474">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002486/index.html#r1c1" target="_new">BGC0002486</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>tolypamide</td>
<td>Tolypothrix sp. PCC 7601</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r259c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-33"></div> <span class="help-tooltip" id="cb-general-help-33">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r259c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r259c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r259c1_1.svg">
NZ_FMXB01000006 (75087-96951): Methanobrevibacter millerae strain DSM 16643, ... (30% of genes show similarity), thioamitides
</option>

<option value = "svg/clusterblast_r259c1_2.svg">
NZ_CP010834 (536116-582695): Methanobrevibacter sp. YE315 chromosome, complet... (20% of genes show similarity), NRPS,thioamitides
</option>

<option value = "svg/clusterblast_r259c1_3.svg">
NZ_RKRG01000004 (69313-91171): Methanobrevibacter gottschalkii DSM 11977 Ga02... (31% of genes show similarity), thioamitides
</option>

<option value = "svg/clusterblast_r259c1_4.svg">
NZ_CP011266 (648255-670119): Methanobrevibacter millerae strain SM9 chromosom... (28% of genes show similarity), thioamitides
</option>

<option value = "svg/clusterblast_r259c1_5.svg">
NZ_HG796203 (34272-56151): Methanobrevibacter oralis JMR01, whole genome shot... (30% of genes show similarity), thioamitides
</option>

<option value = "svg/clusterblast_r259c1_6.svg">
NC_021355 (479130-501258): Methanobrevibacter sp. AbM4, complete sequence (38% of genes show similarity), thioamitides
</option>

<option value = "svg/clusterblast_r259c1_7.svg">
NZ_CP081485 (580821-602681): Methanobrevibacter sp. TLL-48-HuF1 chromosome, c... (33% of genes show similarity), thioamitides
</option>

<option value = "svg/clusterblast_r259c1_8.svg">
NZ_CP017803 (326199-348059): Methanobrevibacter smithii strain KB11 chromosom... (29% of genes show similarity), thioamitides
</option>

<option value = "svg/clusterblast_r259c1_9.svg">
NZ_AP025586 (366379-388239): Methanobrevibacter smithii strain CE91-St67 chro... (33% of genes show similarity), thioamitides
</option>

<option value = "svg/clusterblast_r259c1_10.svg">
NC_009515 (443899-465759): Methanobrevibacter smithii ATCC 35061, complete se... (29% of genes show similarity), thioamitides
</option>

</select>
<button id= "clusterblast-r259c1-download">Download graphic</button>
</div>
<div id="clusterblast-r259c1-svg"></div>

</div>
</div>

<div class="body-details-section r259c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-34"></div> <span class="help-tooltip" id="cb-known-help-34">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r259c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-35"></div> <span class="help-tooltip" id="cb-sub-help-35">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r259c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-37"></div> <span class="help-tooltip" id="tfbs-body-help-37">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 18.61 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r259c1-3799-BldD"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r259c1-tfbs-finder" data-name="r259c1-tfbs-finder">
TFBS Finder
</div>


</div>










<div class="sidepanel-details-section r259c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-36"></div> <span class="help-tooltip" id="tfbs-side-help-36">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>





<div class="page" id='r338c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_338_length_5055_cov_56.8 - Region 1 - NRPS,NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-67"></div> <span class="help-tooltip" id="region_svg-help-67">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_338_length_5055_cov_56.8.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r338c1-svg" data-filename="NODE_338_length_5055_cov_56.8_r338c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 5,055 nt. (total: 5,055 nt)

<a class="cluster-rules-header" id="r338c1-rules-header" href="#r338c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r338c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r338c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-68"></div> <span class="help-tooltip" id="focus-panel-help-68">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r338c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r338c1-gene-table-details" data-name="r338c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r338c1-nrps_pks" data-name="r338c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r338c1-MIBiG-cluster-compare" data-name="r338c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r338c1-clusterblast" data-name="r338c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r338c1-knownclusterblast" data-name="r338c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r338c1-subclusterblast" data-name="r338c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r338c1-nrps-pks-bubbles" data-name="r338c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r338c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-47"></div> <span class="help-tooltip" id="gt-help-tooltip-help-47">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r338c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r338c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r338c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r338c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r338c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-39"></div> <span class="help-tooltip" id="nrps-domain-help-39">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r338c1-details-svg" data-filename="NODE_338_length_5055_cov_56.8_r338c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r338c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r338c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-40"></div> <span class="help-tooltip" id="MIBiG-help-40">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r338c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r338c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r338c1"></div>
<div id="comparison-MIBiG-r338c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:1.00, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.43, order:1.00, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000293: 0-12765">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000293/index.html#r1c1" target="_new">BGC0000293</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP</td>

<td>acetylaszonalenin</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.45</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r338c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:1.00, components:0.52)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000293: 0-12765">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000293/index.html#r1c1" target="_new">BGC0000293</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>acetylaszonalenin</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r338c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-41"></div> <span class="help-tooltip" id="cb-general-help-41">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r338c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r338c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r338c1_1.svg">
NC_014393 (2914101-3025608): Clostridium cellulovorans 743B, complete sequence (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r338c1_2.svg">
NZ_CP019717 (2545473-2659423): Paenibacillus larvae subsp. larvae strain Eric... (2% of genes show similarity), NRPS,T1PKS,phosphonate,phosphonate-like
</option>

<option value = "svg/clusterblast_r338c1_3.svg">
NZ_CP061913 (4107552-4228806): Dactylosporangium vinaceum strain NRRL B-16297... (6% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r338c1_4.svg">
NZ_CP049289 (8455472-8561910): Bradyrhizobium sp. 6(2017) strain 1S3 chromoso... (3% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r338c1_5.svg">
NZ_CAKMMF010000008 (63252-157213): Paenibacillus sp. JJ-246 isolate CIP111893... (5% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r338c1_6.svg">
NZ_CP054839 (3804012-3888358): Saccharopolyspora erythraea strain SCSIO 07745... (4% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r338c1_7.svg">
NZ_CP114036 (4305427-4409110): Streptomyces sp. S465 chromosome, complete genome (2% of genes show similarity), NRPS,NRPS-like,arylpolyene
</option>

<option value = "svg/clusterblast_r338c1_8.svg">
NZ_CP076538 (4614489-4742589): Amycolatopsis aidingensis strain YIM 96748 chr... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,arylpolyene
</option>

<option value = "svg/clusterblast_r338c1_9.svg">
NZ_CP011020 (4570388-4619223): Pseudomonas chlororaphis strain UFB2 isolate S... (5% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r338c1_10.svg">
NZ_JHVT01000011 (162139-210990): Pseudomonas umsongensis UNC430CL58Col N519DR... (5% of genes show similarity), NRPS,NRPS-like
</option>

</select>
<button id= "clusterblast-r338c1-download">Download graphic</button>
</div>
<div id="clusterblast-r338c1-svg"></div>

</div>
</div>

<div class="body-details-section r338c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-42"></div> <span class="help-tooltip" id="cb-known-help-42">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r338c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-43"></div> <span class="help-tooltip" id="cb-sub-help-43">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r338c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-46"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-46">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r338c1-domain-bubble-svg" data-filename="NODE_338_length_5055_cov_56.8_r338c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r338c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 5055): interleaved NRPS-like-NRPS
</option>


</select>

<div class="details-svg" id="r338c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r338c1-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r338c1-nrps_pks_products" data-name="r338c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r338c1-nrps_pks_monomers" data-name="r338c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r338c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-44"></div> <span class="help-tooltip" id="nrps-structures-help-44">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 5055:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r338c1-1" class="smiles-canvas" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Phe)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r338c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(Cc1ccccc1)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r338c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-45"></div> <span class="help-tooltip" id="nrps-monomers-help-45">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">NCFMKG_11720</span></strong>: Phe
<div class="collapser collapser-target-NCFMKG_11720 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (6..372)</strong>: Phe
<div class="collapser collapser-target-NCFMKG_11720 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-NCFMKG_11720-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aromatic side chain (Phe, Trp, Ph-Gly, Tyr, bOH-Tyr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DSATVAAICK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DAWTVAAICK</span> (62% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">NCFMKG_11725</span></strong>: Dab
<div class="collapser collapser-target-NCFMKG_11725 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=dab" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=dab*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (418..811)</strong>: Dab
<div class="collapser collapser-target-NCFMKG_11725 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Dab
<div class="collapser collapser-target-NCFMKG_11725-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIWEITADDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Dab <span class="serif">DIWELTADDK</span> (71% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r381c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> NODE_381_length_4785_cov_56.0 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-69"></div> <span class="help-tooltip" id="region_svg-help-69">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = NODE_381_length_4785_cov_56.0.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r381c1-svg" data-filename="NODE_381_length_4785_cov_56.0_r381c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 4,785 nt. (total: 4,785 nt)

<a class="cluster-rules-header" id="r381c1-rules-header" href="#r381c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r381c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r381c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-70"></div> <span class="help-tooltip" id="focus-panel-help-70">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r381c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r381c1-gene-table-details" data-name="r381c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r381c1-nrps_pks" data-name="r381c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r381c1-MIBiG-cluster-compare" data-name="r381c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r381c1-clusterblast" data-name="r381c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r381c1-knownclusterblast" data-name="r381c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r381c1-subclusterblast" data-name="r381c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r381c1-tfbs-finder" data-name="r381c1-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r381c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-56"></div> <span class="help-tooltip" id="gt-help-tooltip-help-56">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r381c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r381c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r381c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r381c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r381c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-48"></div> <span class="help-tooltip" id="nrps-domain-help-48">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r381c1-details-svg" data-filename="NODE_381_length_4785_cov_56.0_r381c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r381c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r381c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-49"></div> <span class="help-tooltip" id="MIBiG-help-49">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r381c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r381c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r381c1"></div>
<div id="comparison-MIBiG-r381c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP, Saccharide</td>

<td>minimycin</td>


<td>Streptomyces hygroscopicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>pyrrolizixenamide A</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP, Alkaloid</td>

<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>AbT1</td>


<td>Aureobasidium pullulans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308: 0-7851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>aureusimine A, aureusimine B, aureusimine C</td>


<td>Staphylococcus aureus subsp. aureus str. JKD6008</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP, Polyketide</td>

<td>myxoprincomide-c506</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001834: 523-33403">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001834/index.html#r1c1" target="_new">BGC0001834</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>NRP</td>

<td>argyrin</td>


<td>Cystobacter sp. SBCb004</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r381c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002295: 0-9315">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002295/index.html#r1c1" target="_new">BGC0002295</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP, Saccharide</td>
<td>minimycin</td>
<td>Streptomyces hygroscopicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>pyrrolizixenamide A</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP, Alkaloid</td>
<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>AbT1</td>
<td>Aureobasidium pullulans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000308: 0-7851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000308/index.html#r1c1" target="_new">BGC0000308</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.44, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>NRP</td>
<td>aureusimine A, aureusimine B, aureusimine C</td>
<td>Staphylococcus aureus subsp. aureus str. JKD6008</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000393: 0-47918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000393/index.html#r1c1" target="_new">BGC0000393</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP, Polyketide</td>
<td>myxoprincomide-c506</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001834: 523-33403">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001834/index.html#r1c1" target="_new">BGC0001834</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>NRP</td>
<td>argyrin</td>
<td>Cystobacter sp. SBCb004</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r381c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-50"></div> <span class="help-tooltip" id="cb-general-help-50">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<p>No significant ClusterBlast hits found.</p>
</div>

</div>
</div>

<div class="body-details-section r381c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-51"></div> <span class="help-tooltip" id="cb-known-help-51">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r381c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-52"></div> <span class="help-tooltip" id="cb-sub-help-52">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r381c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-55"></div> <span class="help-tooltip" id="tfbs-body-help-55">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
OsdR
<div class="collapser collapser-target-OsdR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and stress management regulator<br>
<span>Score</span>: 18.41 out of 30.12
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r381c1-1181-OsdR"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r381c1-nrps_pks_monomers" data-name="r381c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r381c1-tfbs-finder" data-name="r381c1-tfbs-finder">
TFBS Finder
</div>


</div>










<div class="sidepanel-details-section r381c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-53"></div> <span class="help-tooltip" id="nrps-monomers-help-53">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">NCFMKG_12680</span></strong>: X
<div class="collapser collapser-target-NCFMKG_12680 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (24..265)</strong>: X
<div class="collapser collapser-target-NCFMKG_12680 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-NCFMKG_12680-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>  <strong>NOTE: uncertain match</strong><br>
<dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">AI-MDC-IAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r381c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-54"></div> <span class="help-tooltip" id="tfbs-side-help-54">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/OsdR" target="_blank">OsdR</a></td>
<td>Development and stress management regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>